--
-- PostgreSQL database dump
--

-- Dumped from database version 13.1 (Debian 13.1-1.pgdg100+1)
-- Dumped by pg_dump version 13.1 (Debian 13.1-1.pgdg100+1)

SET statement_timeout = 0;
SET lock_timeout = 0;
SET idle_in_transaction_session_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SELECT pg_catalog.set_config('search_path', '', false);
SET check_function_bodies = false;
SET xmloption = content;
SET client_min_messages = warning;
SET row_security = off;

SET default_tablespace = '';

SET default_table_access_method = heap;

--
-- Name: scopus; Type: TABLE; Schema: public; Owner: postgres
--

CREATE TABLE public.scopus (
    id integer NOT NULL,
    author text,
    title text,
    doi text,
    year text,
    abstract text,
    url text
);


ALTER TABLE public.scopus OWNER TO postgres;

--
-- Name: scopus_id_seq; Type: SEQUENCE; Schema: public; Owner: postgres
--

CREATE SEQUENCE public.scopus_id_seq
    AS integer
    START WITH 1
    INCREMENT BY 1
    NO MINVALUE
    NO MAXVALUE
    CACHE 1;


ALTER TABLE public.scopus_id_seq OWNER TO postgres;

--
-- Name: scopus_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
--

ALTER SEQUENCE public.scopus_id_seq OWNED BY public.scopus.id;


--
-- Name: scopus id; Type: DEFAULT; Schema: public; Owner: postgres
--

ALTER TABLE ONLY public.scopus ALTER COLUMN id SET DEFAULT nextval('public.scopus_id_seq'::regclass);


--
-- Data for Name: scopus; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.scopus (id, author, title, doi, year, abstract, url) FROM stdin;
1	Klymenko, M.V. and Vaitkus, J.A. and Smith, J.S. and Cole, J.H.	NanoNET: An extendable Python framework for semi-empirical tight-binding models	10.1016/j.cpc.2020.107676	2021		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85093703421&doi=10.1016%2fj.cpc.2020.107676&partnerID=40&md5=64cae03ab9761ddb74469ce7662352ea
2	Jiang, J. and Wu, Q. and Cao, J. and Xia, X. and Zhang, L.	Recommending tags for pull requests in GitHub	10.1016/j.infsof.2020.106394	2021		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85092068193&doi=10.1016%2fj.infsof.2020.106394&partnerID=40&md5=4ee8a71e5e67bc61a40e84464506400e
3	Vandenbon, A. and Diez, D.	A clustering-independent method for finding differentially expressed genes in single-cell transcriptome data	10.1038/s41467-020-17900-3	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85089959492&doi=10.1038%2fs41467-020-17900-3&partnerID=40&md5=8689d80ad95d749b8647b6aa1702553e
4	Gousiya Begum and Huq, S.Z.U. and Kumar, A.P.S.	Sandbox security model for Hadoop file system	10.1186/s40537-020-00356-z	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85091723500&doi=10.1186%2fs40537-020-00356-z&partnerID=40&md5=0544c228f4c502091d39e636cc731f2e
5	Dong, X. and Zhang, L. and Hao, X. and Wang, T. and Vijg, J.	SCCNV: A Software Tool for Identifying Copy Number Variation From Single-Cell Whole-Genome Sequencing	10.3389/fgene.2020.505441	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85096911460&doi=10.3389%2ffgene.2020.505441&partnerID=40&md5=53744123682c18ef81098aaf05f6fd3f
6	Miyaki, R. and Yoshida, N. and Tsuzuki, N. and Yamamoto, R. and Takada, H.	Fuzz4B: A front-end to AFL not only for fuzzing experts	10.1145/3412452.3423572	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85096991783&doi=10.1145%2f3412452.3423572&partnerID=40&md5=016f6557b31fc37fb2e5b5f5c3165ccc
7	Fischbach, A. and Strohschein, J. and Bunte, A. and Stork, J. and Faeskorn-Woyke, H. and Moriz, N. and Bartz-Beielstein, T.	CAAI—a cognitive architecture to introduce artificial intelligence in cyber-physical production systems	10.1007/s00170-020-06094-z	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85091780685&doi=10.1007%2fs00170-020-06094-z&partnerID=40&md5=87c2e04e856f2d9703ee6164f64dd45f
8	Zhou, Q. and Wang, Y. and Fan, Y. and Wu, X. and Zhang, S. and Kang, B. and Latecki, L.J.	AGLNet: Towards real-time semantic segmentation of self-driving images via attention-guided lightweight network	10.1016/j.asoc.2020.106682	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85090219040&doi=10.1016%2fj.asoc.2020.106682&partnerID=40&md5=8db4eb8ecc0ee0bf2c360e70736f9e34
9	Fang, Z. and Zhou, H.	Identification of the conjugative and mobilizable plasmid fragments in the plasmidome using sequence signatures	10.1099/mgen.0.000459	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85097002939&doi=10.1099%2fmgen.0.000459&partnerID=40&md5=156831ee64593ad9f8cbe414b208da9c
10	Du, R. and Turner, E. and Dzitsiuk, M. and Prasso, L. and Duarte, I. and Dourgarian, J. and Afonso, J. and Pascoal, J. and Gladstone, J. and Cruces, N. and Izadi, S. and Kowdle, A. and Tsotsos, K. and Kim, D.	DepthLab: Real-time 3D interaction with depth maps for mobile augmented reality	10.1145/3379337.3415881	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85095609289&doi=10.1145%2f3379337.3415881&partnerID=40&md5=6e25b17d4c445bcc9d91639d54eecaac
11	Jančigová, I. and Kovalčíková, K. and Weeber, R. and Cimrák, I.	PyOIF: Computational tool for modelling of multi-cell flows in complex geometries	10.1371/journal.pcbi.1008249	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85094677907&doi=10.1371%2fjournal.pcbi.1008249&partnerID=40&md5=2d68172d8b9e0f40bc87df4d5625691c
12	Scoccia, G.L. and Autili, M.	Web frameworks for desktop apps: An exploratory study	10.1145/3382494.3422171	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85095841232&doi=10.1145%2f3382494.3422171&partnerID=40&md5=9fc34f2c245e192caa30de19a532d5ec
13	Jiang, J. and Chen, X. and Li, T. and Wang, C. and Shen, T. and Zhao, S. and Cui, H. and Wang, C.-L. and Zhang, F.	Uranus: Simple, Efficient SGX Programming and its Applications	10.1145/3320269.3384763	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85090417085&doi=10.1145%2f3320269.3384763&partnerID=40&md5=774a4ad80ec09358941282046e889a8a
14	Schubert, R. and Andaleon, A. and Wheeler, H.E.	Comparing local ancestry inference models in populations of two- And three-way admixture	10.7717/peerj.10090	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85093849582&doi=10.7717%2fpeerj.10090&partnerID=40&md5=1c50e36ec05321c6ddb4d134a58755c6
15	Qin, X. and Zhang, Z. and Huang, C. and Dehghan, M. and Zaiane, O.R. and Jagersand, M.	U2-Net: Going deeper with nested U-structure for salient object detection	10.1016/j.patcog.2020.107404	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85084667067&doi=10.1016%2fj.patcog.2020.107404&partnerID=40&md5=3f69209b10cfcb2ad361d52df4bbcf2e
16	Lovelace, R. and Beecham, R. and Heinen, E. and Vidal Tortosa, E. and Yang, Y. and Slade, C. and Roberts, A.	Is the London Cycle Hire Scheme becoming more inclusive? An evaluation of the shifting spatial distribution of uptake based on 70 million trips	10.1016/j.tra.2020.07.017	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85089236805&doi=10.1016%2fj.tra.2020.07.017&partnerID=40&md5=24f02fc14e8d9123d1bf94171a607517
17	Zhang, H. and Dimitrov, D. and Simpson, L. and Plaks, N. and Singh, B. and Penney, S. and Charles, J. and Sheehan, R. and Flammini, S. and Murphy, S. and Landman, A.	A web-based, mobile-responsive application to screen health care workers for COVID-19 symptoms: Rapid design, deployment, and usage	10.2196/19533	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85096939020&doi=10.2196%2f19533&partnerID=40&md5=b8f674188494f3d142fadc8a0a8c82c3
18	Guebila, M.B.	VFFVA: Dynamic load balancing enables large-scale flux variability analysis	10.1186/s12859-020-03711-2	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85092396672&doi=10.1186%2fs12859-020-03711-2&partnerID=40&md5=f83340abfbc487711f4451c1ad945a7c
19	Cuculovic, M. and Fondement, F. and Devanne, M. and Weber, J. and Hassenforder, M.	Change Detection on JATS Academic Articles: An XML Diff Comparison Study	10.1145/3395027.3419581	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85093094261&doi=10.1145%2f3395027.3419581&partnerID=40&md5=688a09efda95ffabd75ca8aa8451aa27
20	Walsh, K. and Voineagu, M.A. and Vafaee, F. and Voineagu, I.	TDAview: an online visualization tool for topological data analysis	10.1093/bioinformatics/btaa600	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85097003370&doi=10.1093%2fbioinformatics%2fbtaa600&partnerID=40&md5=28c1442821cb48ffc9b75fc75eac447f
21	Silverj, A. and Rota-Stabelli, O.	On the correct interpretation of similarity index in codon usage studies: Comparison with four other metrics and implications for Zika and West Nile virus	10.1016/j.virusres.2020.198097	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85088387605&doi=10.1016%2fj.virusres.2020.198097&partnerID=40&md5=575da4ab4e27b75884501ce23f06ffe0
22	Lin, C. and Nadi, S. and Khazaei, H.	A Large-scale Data Set and an Empirical Study of Docker Images Hosted on Docker Hub	10.1109/ICSME46990.2020.00043	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85096684066&doi=10.1109%2fICSME46990.2020.00043&partnerID=40&md5=56711d85175bab985a31ae44b97b8f9b
23	Salau, J. and Krieter, J.	Analysing the space-usage-pattern of a cow herd using video surveillance and automated motion detection	10.1016/j.biosystemseng.2020.06.015	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85089190975&doi=10.1016%2fj.biosystemseng.2020.06.015&partnerID=40&md5=30bda1c9b475eca6f59e76b00ca60c29
24	Chen, G. and Pui, C.-W. and Li, H. and Young, E.F.Y.	Dr. CU: Detailed Routing by Sparse Grid Graph and Minimum-Area-Captured Path Search	10.1109/TCAD.2019.2927542	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85077797669&doi=10.1109%2fTCAD.2019.2927542&partnerID=40&md5=59444f948f052fa3d2c03634697df2e5
25	Zheng, T. and Zhu, X. and Zhang, X. and Zhao, Z. and Yi, X. and Wang, J. and Li, H.	A machine learning framework for genotyping the structural variations with copy number variant	10.1186/s12920-020-00733-w	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85090003441&doi=10.1186%2fs12920-020-00733-w&partnerID=40&md5=50c179373193c61151593a6b53d2c30a
26	Utomo, P. and Falahah	Building Serverless Website on GitHub Pages	10.1088/1757-899X/879/1/012077	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85091346359&doi=10.1088%2f1757-899X%2f879%2f1%2f012077&partnerID=40&md5=a42a4e51229c007d6a3f1d1c6a969772
27	Farber, M.	Analyzing the github repositories of research papers	10.1145/3383583.3398578	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85095119274&doi=10.1145%2f3383583.3398578&partnerID=40&md5=ba45665816ab7016bebc6a32f0429031
28	Ekholm, J. and Ohukainen, P. and Kangas, A.J. and Kettunen, J. and Wang, Q. and Karsikas, M. and Khan, A.A. and Kingwell, B.A. and Kähönen, M. and Lehtimäki, T. and Raitakari, O.T. and Järvelin, M.-R. and Meikle, P.J. and Ala-Korpela, M.	EpiMetal: an open-source graphical web browser tool for easy statistical analyses in epidemiology and metabolomics	10.1093/ije/dyz244	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85096151779&doi=10.1093%2fije%2fdyz244&partnerID=40&md5=d80d6725a52b855f2dffa49c9499261f
29	Avigal, Y. and Gao, J. and Wong, W. and Li, K. and Pierroz, G. and Deng, F.S. and Theis, M. and Presten, M. and Goldberg, K.	Simulating Polyculture Farming to Tune Automation Policies for Plant Diversity and Precision Irrigation	10.1109/CASE48305.2020.9216984	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85094116346&doi=10.1109%2fCASE48305.2020.9216984&partnerID=40&md5=72a82965ce03cb4fec3e52149000d832
30	Tong, Y. and Wang, J. and Zheng, T. and Zhang, X. and Xiao, X. and Zhu, X. and Lai, X. and Liu, X.	SETE: Sequence-based Ensemble learning approach for TCR Epitope binding prediction	10.1016/j.compbiolchem.2020.107281	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85087106460&doi=10.1016%2fj.compbiolchem.2020.107281&partnerID=40&md5=e67e6d1c849c4e44dcac40277f6add90
31	Deng, Y. and Xu, X. and Qiu, Y. and Xia, J. and Zhang, W. and Liu, S.	A multimodal deep learning framework for predicting drug-drug interaction events	10.1093/bioinformatics/btaa501	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85089401503&doi=10.1093%2fbioinformatics%2fbtaa501&partnerID=40&md5=5d156e3b9d2fac840aee85fcedd3ff80
32	Petit, R.A. and Read, T.D.	Bactopia: A flexible pipeline for complete analysis of bacterial genomes	10.1128/mSystems.00190-20	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85089832943&doi=10.1128%2fmSystems.00190-20&partnerID=40&md5=5af0fcd140516e60f0f521565b7cedd4
33	Wu, Z. and Serie, D. and Xu, G. and Zou, J.	PB-Net: Automatic peak integration by sequential deep learning for multiple reaction monitoring	10.1016/j.jprot.2020.103820	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85084579710&doi=10.1016%2fj.jprot.2020.103820&partnerID=40&md5=e79796465e7f5b4a831e27c17c02b655
34	Patrick, R. and Humphreys, D.T. and Janbandhu, V. and Oshlack, A. and Ho, J.W.K. and Harvey, R.P. and Lo, K.K.	Sierra: Discovery of differential transcript usage from polyA-captured single-cell RNA-seq data	10.1186/s13059-020-02071-7	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85087727921&doi=10.1186%2fs13059-020-02071-7&partnerID=40&md5=516e4bd1fa7573fa12c21bf501de7096
35	Manikumar, D.V.V.S. and Maheswari, B.U.	Blockchain Based DDoS Mitigation Using Machine Learning Techniques	10.1109/ICIRCA48905.2020.9183092	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85092045962&doi=10.1109%2fICIRCA48905.2020.9183092&partnerID=40&md5=816506c29ef45d1a082ccddaa8d8e028
36	Kronk, C.A. and Dexheimer, J.W.	Development of the Gender, Sex, and Sexual Orientation ontology: Evaluation and workflow	10.1093/jamia/ocaa061	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85088491637&doi=10.1093%2fjamia%2focaa061&partnerID=40&md5=29d675f7c37d97d7c334016af01472a1
37	Ross, J.C. and Nardelli, P. and Onieva, J. and Gerard, S.E. and Harmouche, R. and Okajima, Y. and Diaz, A.A. and Washko, G. and San José Estépar, R.	An open-source framework for pulmonary fissure completeness assessment	10.1016/j.compmedimag.2020.101712	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85080058487&doi=10.1016%2fj.compmedimag.2020.101712&partnerID=40&md5=6bb7de7f1be8e2c49a08424dd025d71d
38	Zheng, H. and Kingsford, C. and Marçais, G.	Improved design and analysis of practical minimizers	10.1093/bioinformatics/btaa472	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85087867285&doi=10.1093%2fbioinformatics%2fbtaa472&partnerID=40&md5=0421716ebf213f88a656a998c7dc463d
39	Wickramarachchi, A. and Mallawaarachchi, V. and Rajan, V. and Lin, Y.	MetaBCC-LR: metagenomics binning by coverage and composition for long reads	10.1093/bioinformatics/btaa441	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85087876955&doi=10.1093%2fbioinformatics%2fbtaa441&partnerID=40&md5=f2854a61f9f5b4b52e237987991f1653
40	Fang, H. and Klug, D. and Lamba, H. and Herbsleb, J. and Vasilescu, B.	Need for Tweet: How Open Source Developers Talk about Their GitHub Work on Twitter	10.1145/3379597.3387466	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85093695561&doi=10.1145%2f3379597.3387466&partnerID=40&md5=6542a7b8519d23eb948394e4ccd487bc
41	Wang, P. and Brown, C. and Jennings, J.A. and Stolee, K.T.	An Empirical Study on Regular Expression Bugs	10.1145/3379597.3387464	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85093687976&doi=10.1145%2f3379597.3387464&partnerID=40&md5=07c4cb7902e5d4c1146b552941809161
42	Romano, A. and Wang, W.	WasmView: Visual testing for webassembly applications	10.1145/3377812.3382155	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85094113445&doi=10.1145%2f3377812.3382155&partnerID=40&md5=89f8e431dbf81cf722ded5255f4d704c
43	Ramanathan, M.K. and Clapp, L. and Barik, R. and Sridharan, M.	Piranha: Reducing feature flag debt at Uber	10.1145/3377813.3381350	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85092548989&doi=10.1145%2f3377813.3381350&partnerID=40&md5=b484d75492bbe3ff8113c96dee58922c
44	Chen, B. and Jiang, Z.J.	Studying the use of java logging utilities in the wild	10.1145/3377811.3380408	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85094325311&doi=10.1145%2f3377811.3380408&partnerID=40&md5=6925359b1718ffc865ae4bcab1443a5b
45	Xu, G. and Wang, Q. and Ma, J.	OPUS-Fold: An Open-Source Protein Folding Framework Based on Torsion-Angle Sampling	10.1021/acs.jctc.0c00186	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85086286110&doi=10.1021%2facs.jctc.0c00186&partnerID=40&md5=f52cf35a50adeb1e8b6b89272e55d60f
46	Dai, X. and Boujut, J.F. and Pourroy, F. and Marin, P.	Correlation between process openness and collaboration tool usage in open source hardware design: an empirical study	10.1007/s12008-019-00632-0	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85076222438&doi=10.1007%2fs12008-019-00632-0&partnerID=40&md5=f7bc3c846d0a7af4500655aca85e4195
47	Preechayasomboon, P. and Rombokas, E.	Negshell casting: 3D-printed structured and sacrificial cores for soft robot fabrication	10.1371/journal.pone.0234354	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85086524529&doi=10.1371%2fjournal.pone.0234354&partnerID=40&md5=31d621367ce70c3b33407774a89eccca
48	Torniainen, J. and Afara, I.O. and Prakash, M. and Sarin, J.K. and Stenroth, L. and Töyräs, J.	Open-source python module for automated preprocessing of near infrared spectroscopic data	10.1016/j.aca.2020.02.030	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85080036102&doi=10.1016%2fj.aca.2020.02.030&partnerID=40&md5=5d6aa44390e5095c9bcb6ee248b09eae
49	Eashaan Rao, A. and Chimalakonda, S.	An Exploratory Study towards Understanding Lambda Expressions in Python	10.1145/3383219.3383255	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85090859602&doi=10.1145%2f3383219.3383255&partnerID=40&md5=7adc2870ee237564a7b2e201fd10082c
50	Wu, L. and Deng, Q. and Xu, Z. and Zhou, S. and Li, C. and Li, Y.-X.	A novel virtual barcode strategy for accurate panel-wide variant calling in circulating tumor DNA	10.1186/s12859-020-3412-2	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85083003516&doi=10.1186%2fs12859-020-3412-2&partnerID=40&md5=4660a91bc4ef91b9fa0b009d26ef5003
51	Zhang, Y. and Wang, H. and Wu, Y. and Hu, D. and Wang, T.	GitHub's milestone tool: A mixed-methods analysis on its use	10.1002/smr.2229	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85075388288&doi=10.1002%2fsmr.2229&partnerID=40&md5=491143dfea220852a144e812cc022ee9
52	Fu, J. and Ke, B. and Dong, S.	LCQS: An efficient lossless compression tool of quality scores with random access functionality	10.1186/s12859-020-3428-7	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85081946500&doi=10.1186%2fs12859-020-3428-7&partnerID=40&md5=1768c5d240ee7408ed0029429d869fde
53	Bright, J.M. and Bai, X. and Zhang, Y. and Sun, X. and Acord, B. and Wang, P.	irradpy: Python package for MERRA-2 download, extraction and usage for clear-sky irradiance modelling	10.1016/j.solener.2020.02.061	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85079861962&doi=10.1016%2fj.solener.2020.02.061&partnerID=40&md5=c2eda03f3cfb64c9e9f29e92617b7b25
54	Kobus, R. and Abuín, J.M. and Müller, A. and Hellmann, S.L. and Pichel, J.C. and Pena, T.F. and Hildebrandt, A. and Hankeln, T. and Schmidt, B.	A big data approach to metagenomics for all-food-sequencing	10.1186/s12859-020-3429-6	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85081900147&doi=10.1186%2fs12859-020-3429-6&partnerID=40&md5=d0defae23d40ecb04a0d4d089f01252e
55	Lu, Z. and Gilchrist, M.A. and Emrich, S.	Analysis of mutation bias in shaping CODON usage bias and its association with gene expression across specie	10.29007/87r9	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85085380619&doi=10.29007%2f87r9&partnerID=40&md5=faec1bfee0e518dd9d8235365f4fa52f
56	Agol, E. and Agol, E. and Luger, R. and Luger, R. and Foreman-Mackey, D.	Analytic Planetary Transit Light Curves and Derivatives for Stars with Polynomial Limb Darkening	10.3847/1538-3881/ab4fee	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85086519933&doi=10.3847%2f1538-3881%2fab4fee&partnerID=40&md5=42f2e59b590c51e92e7fbaa73c3f5dea
57	Vassallo, C. and Panichella, S. and Palomba, F. and Proksch, S. and Gall, H.C. and Zaidman, A.	How developers engage with static analysis tools in different contexts	10.1007/s10664-019-09750-5	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85076207019&doi=10.1007%2fs10664-019-09750-5&partnerID=40&md5=93c776c9292559058c77371ea7856e32
58	Wu, Y. and He, K.	Group Normalization	10.1007/s11263-019-01198-w	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85069694688&doi=10.1007%2fs11263-019-01198-w&partnerID=40&md5=c6cfe24ef06716090dc30309edb838b2
59	Liu, L. and Dou, Q. and Chen, H. and Qin, J. and Heng, P.-A.	Multi-Task Deep Model with Margin Ranking Loss for Lung Nodule Analysis	10.1109/TMI.2019.2934577	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85081671450&doi=10.1109%2fTMI.2019.2934577&partnerID=40&md5=76895bbd7ee993c6372cbc6a4f1acd4a
60	Dhasade, A.B. and Venigalla, A.S.M. and Chimalakonda, S.	Towards Prioritizing GitHub Issues	10.1145/3385032.3385052	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85082666629&doi=10.1145%2f3385032.3385052&partnerID=40&md5=50aff18f28a8dbf386e0c146d1414f94
61	Chang, C.H. and Carpenter, I.L. and Jones, W.B.	The ESIF-HPC-2 benchmark suite	10.1145/3380868.3398200	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85095594507&doi=10.1145%2f3380868.3398200&partnerID=40&md5=f1cd66f40efc0655073cd158522c7cf2
62	Benoit, G. and Mariadassou, M. and Robin, S. and Schbath, S. and Peterlongo, P. and Lemaitre, C.	SimkaMin: Fast and resource frugal de novo comparative metagenomics	10.1093/bioinformatics/btz685	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85080842376&doi=10.1093%2fbioinformatics%2fbtz685&partnerID=40&md5=035569ffed6ee4dce503cd8cc07c06e3
63	Asyrofi, M.H. and Thung, F. and Lo, D. and Jiang, L.	AUSearch: Accurate API Usage Search in GitHub Repositories with Type Resolution	10.1109/SANER48275.2020.9054809	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85083550934&doi=10.1109%2fSANER48275.2020.9054809&partnerID=40&md5=1f7f682d33fd5a9a6cfe657bf5c19cef
64	Farahmand, S. and Riley, T. and Zarringhalam, K.	ModEx: A text mining system for extracting mode of regulation of transcription factor-gene regulatory interaction	10.1016/j.jbi.2019.103353	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85077029336&doi=10.1016%2fj.jbi.2019.103353&partnerID=40&md5=3a81acccae490fcb806f49e3b5399e43
65	Flegr, J. and Tureček, P.	New approach and new permutation tests with R programs for analyses of false-negative-contaminated data in medicine and biology	10.1242/BIO.045948	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85089875568&doi=10.1242%2fBIO.045948&partnerID=40&md5=c6a78593422b7381e69757276107a635
66	Ergasheva, S. and Ivanov, V. and Kruglov, A. and Sadovykh, A. and Succi, G. and Zouev, E.	Analysis of development tool usage in software engineering classes	10.1007/978-3-030-57663-9_19	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85089721050&doi=10.1007%2f978-3-030-57663-9_19&partnerID=40&md5=824b1360db49f472cb83796e7160574b
67	Perez-Jimenez, M. and Ramon-Soria, P. and Arrue, B.C. and Ollero, A.	Hecatonquiros: Open-source hardware for aerial manipulation applications	10.1177/1729881420921622	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85084611037&doi=10.1177%2f1729881420921622&partnerID=40&md5=4314299cf7529a547f77a055253f2baa
68	Le, C.P. and Pham, A.Q. and La, H.M. and Feil-Seifer, D.	A Multi-Robotic System for Environmental Dirt Cleaning	10.1109/SII46433.2020.9026295	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85082626697&doi=10.1109%2fSII46433.2020.9026295&partnerID=40&md5=26356419229313917f389e037e95242b
69	Gur, T.	Biobtree: A tool to search and map bioinformatics identifiers and special keywords	10.12688/f1000research.17927.1	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85075964883&doi=10.12688%2ff1000research.17927.1&partnerID=40&md5=cd394940fe3e53245fc22bb98e2ced0f
71	Anwar, A. and Raychowdhury, A.	Autonomous Navigation via Deep Reinforcement Learning for Resource Constraint Edge Nodes Using Transfer Learning	10.1109/ACCESS.2020.2971172	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85081097413&doi=10.1109%2fACCESS.2020.2971172&partnerID=40&md5=c9edfb7317ea58f07fa7e8332a2e7096
72	Imran, A. and Kosar, T.	The impact of auto-refactoring code smells on the resource utilization of cloud software	10.18293/SEKE2020-138	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85090511559&doi=10.18293%2fSEKE2020-138&partnerID=40&md5=1c423914737262b17c8a5a3d5bd50732
73	Unnisa, Z. and Zia, S. and Butt, U.M. and Letchmunan, S. and Ilyas, S.	Ensemble Usage for Classification of EEG Signals A Review with Comparison	10.1007/978-3-030-50353-6_14	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85089238420&doi=10.1007%2f978-3-030-50353-6_14&partnerID=40&md5=007fd5c3c3c14e9f2ef8bd9d103d6c2f
74	Zhu, Z. and Xu, Z. and You, A. and Bai, X.	Semantically multi-modal image synthesis	10.1109/CVPR42600.2020.00551	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85094810175&doi=10.1109%2fCVPR42600.2020.00551&partnerID=40&md5=b23c854135a51da3fa6f02e6db0c6989
75	Maximov, M. and Elezi, I. and Leal-Taixe, L.	CIAGAN: Conditional identity anonymization generative adversarial networks	10.1109/CVPR42600.2020.00549	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85094811356&doi=10.1109%2fCVPR42600.2020.00549&partnerID=40&md5=528b5f33b73d9814a27d68291fa48aa4
76	Pradel, C. and Sileo, D. and Rodrigo, Á. and Peñas, A. and Agirre, E.	Question Answering When Knowledge Bases are Incomplete	10.1007/978-3-030-58219-7_4	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85092166275&doi=10.1007%2f978-3-030-58219-7_4&partnerID=40&md5=785dc7fc6c1c743d292949d30ed908b4
77	You, Y. and Chen, T. and Wang, Z. and Shen, Y.	L2-GCN: Layer-wise and learned efficient training of graph convolutional networks	10.1109/CVPR42600.2020.00220	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85094822233&doi=10.1109%2fCVPR42600.2020.00220&partnerID=40&md5=aa11467ff581cb46bef0fcc08ee2120f
78	Kovaltsuk, A. and Raybould, M.I.J. and Wong, W.K. and Marks, C. and Kelm, S. and Snowden, J. and Trück, J. and Deane, C.M.	Structural diversity of B-cell receptor repertoires along the B-cell differentiation axis in humans and mice	10.1371/journal.pcbi.1007636	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85081072062&doi=10.1371%2fjournal.pcbi.1007636&partnerID=40&md5=63eaf01712971c8e2e7bacb135e8fe88
79	Diamantopoulos, T. and Symeonidis, A.L.	Mining source code for component reuse	10.1007/978-3-030-30106-4_6	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85083969504&doi=10.1007%2f978-3-030-30106-4_6&partnerID=40&md5=08742a34386a395666bea2be6dec1ef8
80	Debe, M. and Salah, K. and Rehman, M.H.U. and Svetinovic, D.	Blockchain-based decentralized reverse bidding in fog computing	10.1109/ACCESS.2020.2991261	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85084949230&doi=10.1109%2fACCESS.2020.2991261&partnerID=40&md5=71307b2343bbd6087c72d281ac37d43d
82	Zhou, H. and Xu, X. and Lin, D. and Wang, X. and Liu, Z.	Sep-Stereo: Visually Guided Stereophonic Audio Generation by Associating Source Separation	10.1007/978-3-030-58610-2_4	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85093121743&doi=10.1007%2f978-3-030-58610-2_4&partnerID=40&md5=d122e72dccba7f9fc6bede63d68e2216
83	Ma, S.-P. and Hsu, M.-J. and Chen, H.-J. and Su, Y.-S.	API Prober – A Tool for Analyzing Web API Features and Clustering Web APIs	10.1007/978-3-030-34986-8_6	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85083439887&doi=10.1007%2f978-3-030-34986-8_6&partnerID=40&md5=3f257e8065caab9d6111cd33c9900c84
84	Joppich, M. and Olenchuk, M. and Mayer, J.M. and Emslander, Q. and Jimenez-Soto, L.F. and Zimmer, R.	SEQU-INTO: Early detection of impurities, contamination and off-targets (ICOs) in long read/MinION sequencing	10.1016/j.csbj.2020.05.014	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85086568936&doi=10.1016%2fj.csbj.2020.05.014&partnerID=40&md5=9d4d50c963b330be0b613b382d3aaf09
85	Dowling, A. and Swiatowicz, F. and Liu, Y. and Tolnai, A.J. and Engel, F.H.	COMBS: First Open-Source Based Benchmark Suite for Multi-physics Simulation Relevant HPC Research	10.1007/978-3-030-60245-1_1	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85092677337&doi=10.1007%2f978-3-030-60245-1_1&partnerID=40&md5=faca6b4880c353f28188af80e226079e
86	Akatsu, S. and Masuda, A. and Shida, T. and Tsuda, K.	A study of quality indicator model of large-scale open source software projects for adoption decision-making	10.1016/j.procs.2020.09.020	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85093365060&doi=10.1016%2fj.procs.2020.09.020&partnerID=40&md5=44c68dc4d7e58aa73b2936215b8cdaf6
87	Zhang, L. and Tanno, R. and Bronik, K. and Jin, C. and Nachev, P. and Barkhof, F. and Ciccarelli, O. and Alexander, D.C.	Learning to segment when experts disagree	10.1007/978-3-030-59710-8_18	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85093116546&doi=10.1007%2f978-3-030-59710-8_18&partnerID=40&md5=fba2d3b1132d07ddccf468a3d64935a9
88	Huang, Y. and Chen, J. and Ouyang, W. and Wan, W. and Xue, Y.	Image Captioning with End-to-End Attribute Detection and Subsequent Attributes Prediction	10.1109/TIP.2020.2969330	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85079506459&doi=10.1109%2fTIP.2020.2969330&partnerID=40&md5=624ea52ee7fa2479d7ea6675bacc9f21
89	Seifert, H. and Weber, M. and Glöckner, F.O. and Kostadinov, I.	An open-source GIS-enabled lookup service for Nagoya Protocol party information	10.1093/database/baaa014	2020		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85083366543&doi=10.1093%2fdatabase%2fbaaa014&partnerID=40&md5=68bd8c43777fd1f347041df576d1cfb6
90	An, L. and Yang, T. and Yang, J. and Nuebler, J. and Xiang, G. and Hardison, R.C. and Li, Q. and Zhang, Y.	OnTAD: Hierarchical domain structure reveals the divergence of activity among TADs and boundaries	10.1186/s13059-019-1893-y	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85076860203&doi=10.1186%2fs13059-019-1893-y&partnerID=40&md5=1effa847ee8d4183cd1387dba344fce5
91	Usevich, K. and Markovsky, I.	Software package for mosaic-Hankel structured low-rank approximation	10.1109/CDC40024.2019.9028867	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85082477622&doi=10.1109%2fCDC40024.2019.9028867&partnerID=40&md5=c9d568596110327c0811b1b6e374c407
92	Masoudi-Sobhanzadeh, Y. and Omidi, Y. and Amanlou, M. and Masoudi-Nejad, A.	Trader as a new optimization algorithm predicts drug-target interactions efficiently	10.1038/s41598-019-45814-8	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85068039947&doi=10.1038%2fs41598-019-45814-8&partnerID=40&md5=3990b19e3b6f40138e48e65e02fad041
93	Lin, S. and Ji, R. and Chen, C. and Tao, D. and Luo, J.	Holistic CNN Compression via Low-Rank Decomposition with Knowledge Transfer	10.1109/TPAMI.2018.2873305	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85054402256&doi=10.1109%2fTPAMI.2018.2873305&partnerID=40&md5=36f9680975dd44383f4b33ef91f67dc9
94	Wang, Z. and Mamun, A.-A. and Cai, X. and Ravishanker, N. and Rajasekaran, S.	Efficient sequential and parallel algorithms for estimating higher order spectra	10.1145/3357384.3358062	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85075422530&doi=10.1145%2f3357384.3358062&partnerID=40&md5=66b6b59cfef3804ad6b28a08bc6e9161
95	Escobar-Velasquez, C. and Osorio-Riano, M. and Linares-Vasquez, M.	MutAPK: Source-codeless mutant generation for android apps	10.1109/ASE.2019.00109	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85078925599&doi=10.1109%2fASE.2019.00109&partnerID=40&md5=d6a858661dbffa78d06a499588152c84
96	Peterson, T.J. and Fulton, S.	Joint Estimation of Gross Recharge, Groundwater Usage, and Hydraulic Properties within HydroSight	10.1111/gwat.12946	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85074279030&doi=10.1111%2fgwat.12946&partnerID=40&md5=c864a3fab8e9916649259977668ffeb0
97	Li, D. and Wang, H. and Xu, C. and Zhang, R. and Cheung, S.-C. and Ma, X.	SGUARD: A feature-based clustering tool for effective spreadsheet defect detection	10.1109/ASE.2019.00122	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85078947024&doi=10.1109%2fASE.2019.00122&partnerID=40&md5=313e083c5b4cd6af26f6197f91735b32
98	Athron, P. and Balázs, C. and Bardsley, M. and Fowlie, A. and Harries, D. and White, G.	Bubbleprofiler: Finding the field profile and action for cosmological phase transitions	10.1016/j.cpc.2019.05.017	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85066957132&doi=10.1016%2fj.cpc.2019.05.017&partnerID=40&md5=0d2c5ebfbca43bfece17e30e0dd0c23f
99	Tegunov, D. and Cramer, P.	Real-time cryo-electron microscopy data preprocessing with Warp	10.1038/s41592-019-0580-y	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85074279272&doi=10.1038%2fs41592-019-0580-y&partnerID=40&md5=978d973acb4c51c36cb0b726cd52a8b2
100	Si, D. and Cheng, S.C. and Xing, R. and Liu, C. and Wu, H.Y.	Scaling up prediction of psychosis by natural language processing	10.1109/ICTAI.2019.00055	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85081080446&doi=10.1109%2fICTAI.2019.00055&partnerID=40&md5=e4f35e0ddbf13ca55df454dd9e320e1e
101	Li, C. and Zhou, M. and Gu, Z. and Gu, M. and Zhang, H.	Ares: Inferring error specifications through static analysis	10.1109/ASE.2019.00130	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85078941395&doi=10.1109%2fASE.2019.00130&partnerID=40&md5=e13f9659a6f99a8476d6c33ae99e141f
102	An, S. and Che, G. and Zhou, F. and Liu, X. and Ma, X. and Chen, Y.	Fast and Incremental Loop Closure Detection Using Proximity Graphs	10.1109/IROS40897.2019.8968043	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85081159056&doi=10.1109%2fIROS40897.2019.8968043&partnerID=40&md5=f1afcbc34a8eaf3d8a1e14c3c9ffe672
103	Khaled Saifullah, C.M. and Asaduzzaman, M. and Roy, C.K.	Learning from examples to find fully qualified names of API elements in code snippets	10.1109/ASE.2019.00032	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85078903099&doi=10.1109%2fASE.2019.00032&partnerID=40&md5=c0b0d2da6a0e2699222a1f48593108df
104	Tripathi, A. and Domingue, B.	Curve fitting from probabilistic emissions and applications to dynamic item response theory	10.1109/ICDM.2019.00170	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85078870971&doi=10.1109%2fICDM.2019.00170&partnerID=40&md5=44a508647a346143954448bd90432a03
105	Warren, R.L. and Coombe, L. and Mohamadi, H. and Zhang, J. and Jaquish, B. and Isabel, N. and Jones, S.J.M. and Bousquet, J. and Bohlmann, J.	ntEdit: scalable genome sequence polishing	10.1093/bioinformatics/btz400	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85074306724&doi=10.1093%2fbioinformatics%2fbtz400&partnerID=40&md5=61976839122ab36abd940a3aa615ce0c
106	Seifer, P. and Härtel, J. and Leinberger, M. and Lämmel, R. and Staab, S.	Empirical study on the usage of graph query languages in open source Java projects	10.1145/3357766.3359541	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85076766330&doi=10.1145%2f3357766.3359541&partnerID=40&md5=6765027b0204dbc000e2b28bf1152abf
107	Hoffmann, N. and Hartler, J. and Ahrends, R.	JmzTab-M: A Reference Parser, Writer, and Validator for the Proteomics Standards Initiative mzTab 2.0 Metabolomics Standard	10.1021/acs.analchem.9b01987	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85072971699&doi=10.1021%2facs.analchem.9b01987&partnerID=40&md5=49a9fd48f466ac039cd35b0319df65f6
108	Yao, W. and Huang, F. and Zhang, X. and Tang, J.	ECOGEMS: Efficient compression and retrieve of SNP data of 2058 rice accessions with integer sparse matrices	10.1093/bioinformatics/btz186	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85073176664&doi=10.1093%2fbioinformatics%2fbtz186&partnerID=40&md5=0b348a7613961ca1630bb8c335775419
109	Chen, Y.-A. and Tripathi, L.P. and Fujiwara, T. and Kameyama, T. and Itoh, M.N. and Mizuguchi, K.	The TargetMine Data Warehouse: Enhancement and Updates	10.3389/fgene.2019.00934	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85074130327&doi=10.3389%2ffgene.2019.00934&partnerID=40&md5=838f1aa3d5adc015426def1966a75b42
110	Hu, X. and Friedberg, I.	SwiftOrtho: A fast, memory-efficient, multiple genome orthology classifier	10.1093/gigascience/giz118	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85074089972&doi=10.1093%2fgigascience%2fgiz118&partnerID=40&md5=2edd399fa50a6217000c0f2cd45a8c81
111	Tang, Y. and Liu, X.	G2P: A Genome-Wide-Association-Study simulation tool for genotype simulation, phenotype simulation and power evaluation	10.1093/bioinformatics/btz126	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85067889548&doi=10.1093%2fbioinformatics%2fbtz126&partnerID=40&md5=fc7b9b31752f6ca22fe16be6b224ef05
112	Mei, K. and Li, J. and Zhang, J. and Wu, H. and Li, J. and Huang, R.	HighEr-resolution network for image demosaicing and enhancing	10.1109/ICCVW.2019.00427	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85082468300&doi=10.1109%2fICCVW.2019.00427&partnerID=40&md5=38a840e866d793c8b2cbc9ad2a18e0c6
113	Schorghuber, M. and Steininger, D. and Cabon, Y. and Humenberger, M. and Gelautz, M.	SLAMANTIC - Leveraging semantics to improve VSLAM in dynamic environments	10.1109/ICCVW.2019.00468	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85082464315&doi=10.1109%2fICCVW.2019.00468&partnerID=40&md5=c4ebb1cc5b44b7e7301b995d779e1634
114	Hu, S.X. and Zagoruyko, S. and Komodakis, N.	Exploring weight symmetry in deep neural networks	10.1016/j.cviu.2019.07.006	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85070393548&doi=10.1016%2fj.cviu.2019.07.006&partnerID=40&md5=9644659904218a4aa9452344332f6193
115	Aguilera, L.U. and Raymond, W. and Fox, Z.R. and May, M. and Djokic, E. and Morisaki, T. and Stasevich, T.J. and Munsky, B.	Computational design and interpretation of single-RNA translation experiments	10.1371/journal.pcbi.1007425	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85074242850&doi=10.1371%2fjournal.pcbi.1007425&partnerID=40&md5=efad652c8921ed7c13b560ea30262fa3
116	Huang, Z. and Wang, X. and Huang, L. and Huang, C. and Wei, Y. and Liu, W.	CCNet: Criss-cross attention for semantic segmentation	10.1109/ICCV.2019.00069	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85079745733&doi=10.1109%2fICCV.2019.00069&partnerID=40&md5=c9dc5eb6a489aec64a28b5a9e8694702
117	Wan, F. and Zhu, Y. and Hu, H. and Dai, A. and Cai, X. and Chen, L. and Gong, H. and Xia, T. and Yang, D. and Wang, M.-W. and Zeng, J.	DeepCPI: A Deep Learning-based Framework for Large-scale in silico Drug Screening	10.1016/j.gpb.2019.04.003	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85079810153&doi=10.1016%2fj.gpb.2019.04.003&partnerID=40&md5=037432e53a20eb81d3c61d66cb1130fc
118	Hai, J. and Wang, C. and Chen, X. and Li, T.O. and Cui, H. and Wang, S.	Fulva: Efficient live migration for in-memory key-value stores with zero downtime	10.1109/SRDS47363.2019.00019	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85084126416&doi=10.1109%2fSRDS47363.2019.00019&partnerID=40&md5=df57edba8f7f263d69ebb0100b2552fd
119	Borges, H. and Brito, R. and Tulio Valente, M.	Beyond textual issues: Understanding the usage and impact of GitHub reactions	10.1145/3350768.3350788	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85073188315&doi=10.1145%2f3350768.3350788&partnerID=40&md5=b64a53e9da8ba76c99d708395efbed3d
120	Fazzolino, R. and Rodrigues, G.N.	Feature-trace: Generating operational profile and supporting testing prioritization from BDD features	10.1145/3350768.3350781	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85073209112&doi=10.1145%2f3350768.3350781&partnerID=40&md5=43067642556626381dc7d258bb643e28
121	Tyryshkina, A. and Coraor, N. and Nekrutenko, A.	Predicting runtimes of bioinformatics tools based on historical data: Five years of Galaxy usage	10.1093/bioinformatics/btz054	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85072508575&doi=10.1093%2fbioinformatics%2fbtz054&partnerID=40&md5=91a23e21ed94c04f1909f1ade57cd772
122	Mukherjee, K. and Alipanahi, B. and Kahveci, T. and Salmela, L. and Boucher, C.	Aligning optical maps to de Bruijn graphs	10.1093/bioinformatics/btz069	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85072332120&doi=10.1093%2fbioinformatics%2fbtz069&partnerID=40&md5=37c3de3d6c8d1e12124dbd224fd60d7b
123	Pereira-Flores, E. and Glöckner, F.O. and Fernandez-Guerra, A.	Fast and accurate average genome size and 16S rRNA gene average copy number computation in metagenomic data	10.1186/s12859-019-3031-y	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85071767766&doi=10.1186%2fs12859-019-3031-y&partnerID=40&md5=9da4eabc4fbcf8d25ca829ebfaa2eb9d
124	Kallis, R. and Di Sorbo, A. and Canfora, G. and Panichella, S.	Ticket Tagger: Machine Learning Driven Issue Classification	10.1109/ICSME.2019.00070	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85077205637&doi=10.1109%2fICSME.2019.00070&partnerID=40&md5=7a617f0a31e3b6be8984d97167e11370
125	Bradley, K. and Godfrey, M.	A study on the effects of exception usage in open-source C++ systems	10.1109/SCAM.2019.00010	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85077812575&doi=10.1109%2fSCAM.2019.00010&partnerID=40&md5=a891cdbd21df4f81fe61a52e8798ea54
126	Uthayamoorthy, K. and Kanthasamy, K. and Senthaalan, T. and Sarveswaran, K. and Dias, G.	DDSpell - A Data Driven Spell Checker and Suggestion Generator for the Tamil Language	10.1109/ICTer48817.2019.9023698	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85082302525&doi=10.1109%2fICTer48817.2019.9023698&partnerID=40&md5=eca7b16ee37b8adbaef086b1eab2ffb5
127	Avelino, G. and Constantinou, E. and Valente, M.T. and Serebrenik, A.	On the abandonment and survival of open source projects: An empirical investigation	10.1109/ESEM.2019.8870181	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85072031326&doi=10.1109%2fESEM.2019.8870181&partnerID=40&md5=b3a793b07964f4d7b3ce3d2f6a7b515f
128	Georgeson, P. and Syme, A. and Sloggett, C. and Chung, J. and Dashnow, H. and Milton, M. and Lonsdale, A. and Powell, D. and Seemann, T. and Pope, B.	Bionitio: Demonstrating and facilitating best practices for bioinformatics command-line software	10.1093/gigascience/giz109	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85072574632&doi=10.1093%2fgigascience%2fgiz109&partnerID=40&md5=3855835094bd0f75ba052e56a9a19573
129	Borg, M. and Svensson, O. and Berg, K. and Hansson, D.	SZZ unleashed: An open implementation of the SZZ algorithm-featuring example usage in a study of just-in-time bug prediction for the jenkins project	10.1145/3340482.3342742	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85076393558&doi=10.1145%2f3340482.3342742&partnerID=40&md5=d52bd5d7284bb13a4dcfe42e503eceef
130	Steenwyk, J.L. and Rokas, A.	Treehouse: A user-friendly application to obtain subtrees from large phylogenies	10.1186/s13104-019-4577-5	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85071577517&doi=10.1186%2fs13104-019-4577-5&partnerID=40&md5=62c66e48329971da8f3516dff603335a
131	Abid, S.	Recommending related functions from API usage-based function clone structures	10.1145/3338906.3342486	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85071903250&doi=10.1145%2f3338906.3342486&partnerID=40&md5=12b29c2cf6fa6827474edc9ffa8e0719
132	Islam, M.J. and Nguyen, G. and Pan, R. and Rajan, H.	A comprehensive study on deep learning bug characteristics	10.1145/3338906.3338955	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85071909185&doi=10.1145%2f3338906.3338955&partnerID=40&md5=5417703fdc2f3e7bbab976c1cfa80ea4
133	Chen, Z. and Cao, Y. and Lu, X. and Mei, Q. and Liu, X.	SEntiMoji: An emoji-powered learning approach for sentiment analysis in software engineering	10.1145/3338906.3338977	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85071906723&doi=10.1145%2f3338906.3338977&partnerID=40&md5=1e967ded2d75e2d58c7afc55754eefc0
134	Sun, Y. and Venugopal, V. and Brandt, A.R.	Short-term solar power forecast with deep learning: Exploring optimal input and output configuration	10.1016/j.solener.2019.06.041	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85067921025&doi=10.1016%2fj.solener.2019.06.041&partnerID=40&md5=a1a0f73ffd0db8dfecadd2120d671a3f
135	Torner, J. and Skouras, S. and Molinuevo, J.L. and Gispert, J.D. and Alpiste, F.	Multipurpose Virtual Reality Environment for Biomedical and Health Applications	10.1109/TNSRE.2019.2926786	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85071348830&doi=10.1109%2fTNSRE.2019.2926786&partnerID=40&md5=5329da375e5f1c030800b2906fd911cb
136	Tian, M. and Guo, H. and Chen, H. and Wang, Q. and Long, C. and Ma, Y.	Automated pig counting using deep learning	10.1016/j.compag.2019.05.049	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85067302123&doi=10.1016%2fj.compag.2019.05.049&partnerID=40&md5=c42aa19c9a7110c63af4996c90651a84
137	Ren, S. and Jia, M.	Scenario Oriented Program Slicing for Large-Scale Software Through Constraint Logic Programming and Program Transformation	10.1109/ACCESS.2018.2853153	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85049770937&doi=10.1109%2fACCESS.2018.2853153&partnerID=40&md5=a68a76b5e162176fcd196c390614dc97
138	Zou, W. and Zhang, W. and Xia, X. and Holmes, R. and Chen, Z.	Branch Use in Practice: A Large-Scale Empirical Study of 2,923 Projects on GitHub	10.1109/QRS.2019.00047	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85073782631&doi=10.1109%2fQRS.2019.00047&partnerID=40&md5=c4f5acf40f44203ca6fdf2e1d82564ec
139	Grinvald, M. and Furrer, F. and Novkovic, T. and Chung, J.J. and Cadena, C. and Siegwart, R. and Nieto, J.	Volumetric instance-aware semantic mapping and 3D object discovery	10.1109/LRA.2019.2923960	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85068681307&doi=10.1109%2fLRA.2019.2923960&partnerID=40&md5=f18bd3cd788a49420bf21a7b4dc0a958
140	Lakhani, J. and Khunteta, A. and Choudhary, A. and Harwani, D.	MPSAGA: a matrix-based pair-wise sequence alignment algorithm for global alignment with position based sequence representation	10.1007/s12046-019-1141-x	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85068172167&doi=10.1007%2fs12046-019-1141-x&partnerID=40&md5=d612fff71b1cbe91a93810bb47c305fe
141	Hayen, L. and Severijns, N.	Beta Spectrum Generator: High precision allowed β spectrum shapes	10.1016/j.cpc.2019.02.012	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85062808184&doi=10.1016%2fj.cpc.2019.02.012&partnerID=40&md5=ed913caa0bad3c440ccdc55d73c007c7
142	Baltes, S. and Diehl, S.	Usage and attribution of Stack Overflow code snippets in GitHub projects	10.1007/s10664-018-9650-5	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85054385016&doi=10.1007%2fs10664-018-9650-5&partnerID=40&md5=de4d56d866ca9ea5b4a38d350cfa9e83
143	Lapierre, N. and Mangul, S. and Alser, M. and Mandric, I. and Wu, N.C. and Koslicki, D. and Eskin, E.	MiCoP: Microbial community profiling method for detecting viral and fungal organisms in metagenomic samples	10.1186/s12864-019-5699-9	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85066843320&doi=10.1186%2fs12864-019-5699-9&partnerID=40&md5=5ddc3867faf2619a46389a46a63491e2
144	Chu, C. and Li, X. and Wu, Y.	GAPPadder: A sensitive approach for closing gaps on draft genomes with short sequence reads	10.1186/s12864-019-5703-4	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85066843780&doi=10.1186%2fs12864-019-5703-4&partnerID=40&md5=5ddf5e1106134f472393ce4879d6b6da
145	Gao, S. and Zhuang, X.	Multi-scale deep neural networks for real image super-resolution	10.1109/CVPRW.2019.00252	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85078101629&doi=10.1109%2fCVPRW.2019.00252&partnerID=40&md5=8043341b70a3e3216897e18329a729fe
146	Cheng, G. and Matsune, A. and Li, Q. and Zhu, L. and Zang, H. and Zhan, S.	Encoder-decoder residual network for real super-resolution	10.1109/CVPRW.2019.00270	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85082884126&doi=10.1109%2fCVPRW.2019.00270&partnerID=40&md5=80ad03b61778642728b3d5c73ffbb8ad
147	Qiao, S. and Lin, Z. and Zhang, J. and Yuille, A.L.	Neural rejuvenation: Improving deep network training by enhancing computational resource utilization	10.1109/CVPR.2019.00015	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85078783923&doi=10.1109%2fCVPR.2019.00015&partnerID=40&md5=7f1aa8313e974b219ea2f84737990665
148	Muenchow, J. and Schäfer, S. and Krüger, E.	Reviewing qualitative GIS research—Toward a wider usage of open-source GIS and reproducible research practices	10.1111/gec3.12441	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85067387953&doi=10.1111%2fgec3.12441&partnerID=40&md5=ebc8dbf49b298822aabf0b1e0405544f
149	Voicu, T.A. and Al-Ars, Z.	SparkJNI: A Toolchain for Hardware Accelerated Big Data Apache Spark	10.1109/ICBDA.2019.8713201	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85066608307&doi=10.1109%2fICBDA.2019.8713201&partnerID=40&md5=87dbe3d34ef5e1832978953b747077a3
150	Pérez-Rubio, P. and Lottaz, C. and Engelmann, J.C.	FastqPuri: High-performance preprocessing of RNA-seq data	10.1186/s12859-019-2799-0	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85065132299&doi=10.1186%2fs12859-019-2799-0&partnerID=40&md5=25632a3045b34d0ae258708864c1cc59
151	Tsubaki, M. and Mizoguchi, T.	Correction to: Fast and accurate molecular property prediction: Learning atomic interactions and potentials with neural networks (Journal of Physical Chemistry Letters (2018) 9:19 (5733-5741) DOI: 10.1021/acs.jpclett.8b01837)	10.1021/acs.jpclett.9b00301	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85065308996&doi=10.1021%2facs.jpclett.9b00301&partnerID=40&md5=39cc33d2cad890c43ef55d120ce072b4
152	Nguyen, P.T. and Di Rocco, J. and Di Ruscio, D. and Ochoa, L. and Degueule, T. and Di Penta, M.	FOCUS: A Recommender System for Mining API Function Calls and Usage Patterns	10.1109/ICSE.2019.00109	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85064745329&doi=10.1109%2fICSE.2019.00109&partnerID=40&md5=ae00b7042201692d4af3586752a891b7
153	Baltes, S. and Treude, C. and Diehl, S.	SOTorrent: Studying the origin, evolution, and usage of stack overflow code snippets	10.1109/MSR.2019.00038	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85072322327&doi=10.1109%2fMSR.2019.00038&partnerID=40&md5=06eeacd48e5ade4f92c3f6d0ccf23eff
154	Biswas, S. and Islam, M.J. and Huang, Y. and Rajan, H.	Boa meets python: A boa dataset of data science software in python language	10.1109/MSR.2019.00086	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85071933042&doi=10.1109%2fMSR.2019.00086&partnerID=40&md5=29dc9e0e59364610a6c434c555293775
155	Hanussek, M. and Bartusch, F. and Kruger, J. and Kohlbacher, O.	BOOTABLE: Bioinformatics benchmark tool suite	10.1109/CCGRID.2019.00027	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85069536228&doi=10.1109%2fCCGRID.2019.00027&partnerID=40&md5=a3c6cebefffbbb62749cb6b895e5c555
157	Kavaler, D. and Trockman, A. and Vasilescu, B. and Filkov, V.	Tool Choice Matters: JavaScript Quality Assurance Tools and Usage Outcomes in GitHub Projects	10.1109/ICSE.2019.00060	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85071947090&doi=10.1109%2fICSE.2019.00060&partnerID=40&md5=b0e63d9763d8e93ff3718ba7d8c93d3b
158	Lin, Y. and Cheng, S. and Shen, J. and Pantic, M.	MobiFace: A novel dataset for mobile face tracking in the wild	10.1109/FG.2019.8756581	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85070461305&doi=10.1109%2fFG.2019.8756581&partnerID=40&md5=78bb3b253ace68850e0b89cf5f8d6999
159	Scoccia, G.L. and Peruma, A. and Pujols, V. and Christians, B. and Krutz, D.	An empirical history of permission requests and mistakes in open source android apps	10.1109/MSR.2019.00090	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85072334068&doi=10.1109%2fMSR.2019.00090&partnerID=40&md5=bd7f8c960e9440d24b53d8b4c334b689
160	Xu, S. and Dong, Z. and Meng, N.	Meditor: Inference and application of API migration edits	10.1109/ICPC.2019.00052	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85070651329&doi=10.1109%2fICPC.2019.00052&partnerID=40&md5=d611eb7ca3a6e88565560cc1ee0ae01b
161	Iacoangeli, A. and Al Khleifat, A. and Sproviero, W. and Shatunov, A. and Jones, A.R. and Morgan, S.L. and Pittman, A. and Dobson, R.J. and Newhouse, S.J. and Al-Chalabi, A.	DNAscan: Personal computer compatible NGS analysis, annotation and visualisation	10.1186/s12859-019-2791-8	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85064860203&doi=10.1186%2fs12859-019-2791-8&partnerID=40&md5=af5fb841b8d1f724c7203ecd466c9c62
162	Seefried, F. and Schmidt, T. and Reinecke, M. and Heinzlmeir, S. and Kuster, B. and Wilhelm, M.	CiRCus: A Framework to Enable Classification of Complex High-Throughput Experiments	10.1021/acs.jproteome.8b00724	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85063527988&doi=10.1021%2facs.jproteome.8b00724&partnerID=40&md5=8fea405ae664bf72b4929150569ee43a
163	Sun, Z. and Gao, S. and Liu, B. and Wang, Y. and Yang, T. and Cui, B.	Magic Cube Bloom Filter: Answering Membership Queries for Multiple Sets	10.1109/BIGCOMP.2019.8679119	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85064651993&doi=10.1109%2fBIGCOMP.2019.8679119&partnerID=40&md5=78c268d929c31384649d9f6a3aa4b704
164	Ren, X. and Zheng, L. and Zhang, Z.	SSCC: A Novel Computational Framework for Rapid and Accurate Clustering Large-scale Single Cell RNA-seq Data	10.1016/j.gpb.2018.10.003	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85067492639&doi=10.1016%2fj.gpb.2018.10.003&partnerID=40&md5=104dc2d80cec8094a84883a3194a4492
165	Tong, Y. and Yang, D. and Jiang, J. and Gao, S. and Cui, B. and Shi, L. and Li, X.	Coloring embedder: A memory efficient data structure for answering multi-set query	10.1109/ICDE.2019.00105	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85067992865&doi=10.1109%2fICDE.2019.00105&partnerID=40&md5=b74b3b537134bd79225dadf9de44135a
166	da Silva Neto, A.M. and Silva, S.R. and Vendruscolo, M. and Camilloni, C. and Montalvão, R.W.	A superposition free method for protein conformational ensemble analyses and local clustering based on a differential geometry representation of backbone	10.1002/prot.25652	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85061041270&doi=10.1002%2fprot.25652&partnerID=40&md5=fe2e25d93caa6f856ba3322f51604946
167	Wang, P. and Bai, G.R. and Stolee, K.T.	Exploring Regular Expression Evolution	10.1109/SANER.2019.8667972	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85064174666&doi=10.1109%2fSANER.2019.8667972&partnerID=40&md5=4cec9cff3af89379065c46dceb56c0ad
168	Madeiral, F. and Urli, S. and Maia, M. and Monperrus, M.	BEARS: An Extensible Java Bug Benchmark for Automatic Program Repair Studies	10.1109/SANER.2019.8667991	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85064151415&doi=10.1109%2fSANER.2019.8667991&partnerID=40&md5=664ffc6e21723bf279da626dc8662b6c
169	Otlu, B. and Can, T.	JOA: Joint Overlap Analysis of multiple genomic interval sets 08 Information and Computing Sciences 0801 Artificial Intelligence and Image Processing 08 Information and Computing Sciences 0802 Computation Theory and Mathematics	10.1186/s12859-019-2698-4	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85062613795&doi=10.1186%2fs12859-019-2698-4&partnerID=40&md5=35ec15e4970f1db0260c92ae80627a63
170	Chiu, H.-K. and Adeli, E. and Wang, B. and Huang, D.-A. and Niebles, J.C.	Action-agnostic human pose forecasting	10.1109/WACV.2019.00156	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85063581024&doi=10.1109%2fWACV.2019.00156&partnerID=40&md5=7d96a08804e833bda296bfd18cbafbbc
171	Casey, E. and Barnum, S. and Griffith, R. and Snyder, J. and van Beek, H. and Nelson, A.	Corrigendum to ‘Advancing coordinated cyber-investigations and tool interoperability using a community developed specification language’ [Digital Investigation 22C (2017) 14–45](S1742287617301007)(10.1016/j.diin.2017.08.002)	10.1016/j.diin.2018.10.001	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85057261321&doi=10.1016%2fj.diin.2018.10.001&partnerID=40&md5=d56964aaf71a1c9c69fb988e1c309fcc
172	Kobus, R. and Lamoth, A. and Müller, A. and Hundt, C. and Kramer, S. and Schmidt, B.	CuBool: Bit-Parallel Boolean Matrix Factorization on CUDA-Enabled Accelerators	10.1109/PADSW.2018.8644574	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85063338298&doi=10.1109%2fPADSW.2018.8644574&partnerID=40&md5=7642dfffd8d3dc42019f354f7810664d
173	Xiao, A. and Wu, Z. and Dong, S.	Ads-hcspark: A scalable haplotypecaller leveraging adaptive data segmentation to accelerate variant calling on spark	10.1186/s12859-019-2665-0	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85061580507&doi=10.1186%2fs12859-019-2665-0&partnerID=40&md5=5ac4d84837f1ffb8049192785a88c596
174	Uszkoreit, J. and Perez-Riverol, Y. and Eggers, B. and Marcus, K. and Eisenacher, M.	Protein Inference Using PIA Workflows and PSI Standard File Formats	10.1021/acs.jproteome.8b00723	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85060918776&doi=10.1021%2facs.jproteome.8b00723&partnerID=40&md5=65a3399eedbd97aff2630ca4fca4178e
175	Sun, C. and Medvedev, P.	Toward fast and accurate SNP genotyping from whole genome sequencing data for bedside diagnostics	10.1093/bioinformatics/bty641	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85061119596&doi=10.1093%2fbioinformatics%2fbty641&partnerID=40&md5=bc41e205b27b4d2f3f606dddd5d641d3
176	Zhernakov, A.I. and Afonin, A.M. and Gavriliuk, N.D. and Moiseeva, O.M. and Zhukov, V.A.	s-dePooler: Determination of polymorphism carriers from overlapping DNA pools	10.1186/s12859-019-2616-9	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85060292432&doi=10.1186%2fs12859-019-2616-9&partnerID=40&md5=6402bf9af164c5707d0ad0906f3f1279
177	Chen, C.-Y. and Chuang, T.-J.	NCLcomparator: Systematically post-screening non-co-linear transcripts (circular, trans-spliced, or fusion RNAs) identified from various detectors	10.1186/s12859-018-2589-0	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85059503343&doi=10.1186%2fs12859-018-2589-0&partnerID=40&md5=55af031257b41841bc83f5ba6cfcf957
178	Lemay, M.J.	Understanding java usability by mining GitHub repositories	10.4230/OASIcs.PLATEAU.2018.2	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85067886618&doi=10.4230%2fOASIcs.PLATEAU.2018.2&partnerID=40&md5=bbb5fc693f341647d987f2b547f31f82
180	Petke, J. and Brownlee, A.E.I.	Software Improvement with Gin: A Case Study	10.1007/978-3-030-27455-9_14	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85072863413&doi=10.1007%2f978-3-030-27455-9_14&partnerID=40&md5=affbdd02f327c3876dde5b143c42e2c7
182	Kubitza, D.O. and Böckmann, M. and Graux, D.	SemanGit: A Linked Dataset from git	10.1007/978-3-030-30796-7_14	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85081082643&doi=10.1007%2f978-3-030-30796-7_14&partnerID=40&md5=e6aedb452a49c79cc28f1b33573c1a91
183	Mohan, V.S. and Vinayakumar, R. and Sowmya, V. and Soman, K.P.	Deep rectified system for high-speed tracking in images	10.3233/JIFS-169907	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85063456639&doi=10.3233%2fJIFS-169907&partnerID=40&md5=f066b9f86b13d6267826a35253052940
184	Shiau, C.-K. and Huang, J.-H. and Tsai, H.-K.	Catana: A tool for generating comprehensive annotations of alternative transcript events	10.1093/bioinformatics/bty795	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85068492251&doi=10.1093%2fbioinformatics%2fbty795&partnerID=40&md5=4d50c96384000b147a5dc2896d4ab2f6
185	Guidotti, R. and Soldani, J. and Neri, D. and Brogi, A. and Pedreschi, D.	Helping your docker images to spread based on explainable models	10.1007/978-3-030-10997-4_13	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85061153581&doi=10.1007%2f978-3-030-10997-4_13&partnerID=40&md5=70fe9654269fdd75d3fed5fbda578f35
187	Fink, L. and Lee, S.C. and Wu, J.Y. and Liu, X. and Song, T. and Velikova, Y. and Stamminger, M. and Navab, N. and Unberath, M.	LumiPath – Towards Real-Time Physically-Based Rendering on Embedded Devices	10.1007/978-3-030-32254-0_75	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85075667340&doi=10.1007%2f978-3-030-32254-0_75&partnerID=40&md5=d5292b3c86634d41124db5a2daa63efc
188	Sharif, S.M.A. and Mahbootf, M.	Deep HOG: A hybrid model to classify bangla isolated alpha-numerical symbols	10.14311/NNW.2019.29.009	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85070239897&doi=10.14311%2fNNW.2019.29.009&partnerID=40&md5=ca74d2f3733d7cac306f72e505bd0a21
189	Anatskiy, E. and Ryan, D.P. and Grüning, B.A. and Arrigoni, L. and Manke, T. and Bönisch, U.	Parkour LiMs: High-quality sample preparation in next generation sequencing	10.1093/bioinformatics/bty820	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85068582245&doi=10.1093%2fbioinformatics%2fbty820&partnerID=40&md5=fb25d53d31523264060b0186a44b294e
190	Goncharenko, A. and Denisov, A. and Alyamkin, S. and Terentev, E.	Trainable Thresholds for Neural Network Quantization	10.1007/978-3-030-20518-8_26	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85067573043&doi=10.1007%2f978-3-030-20518-8_26&partnerID=40&md5=671b47c82dc642d31fd6199403a04988
193	Liu, Q. and Liu, F. and He, J. and Zhou, M. and Hou, T. and Liu, Y.	VFM: Identification of Bacteriophages from Metagenomic Bins and Contigs Based on Features Related to Gene and Genome Composition	10.1109/ACCESS.2019.2957833	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85077239485&doi=10.1109%2fACCESS.2019.2957833&partnerID=40&md5=0301932b5c898e5587db0d0764daebe6
194	Fadishei, H. and Doustian, S. and Saadati, P.	The Merits of Bitset Compression Techniques for Mining Association Rules from Big Data	10.1007/978-3-030-33495-6_10	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85075823324&doi=10.1007%2f978-3-030-33495-6_10&partnerID=40&md5=b1cb51565360a5c80b524ff031d961ee
197	Wang, A. and Shi, Y. and Yi, X. and Yan, Y. and Liao, C. and de Supinski, B.R.	Ompparser: A standalone and unified openmp parser	10.1007/978-3-030-28596-8_10	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85072855794&doi=10.1007%2f978-3-030-28596-8_10&partnerID=40&md5=326987f6455e85a12df3746eb05d6800
198	Raborn, R.T. and Brendel, V.P.	Using RAMPAGE to identify and annotate promoters in insect genomes	10.1007/978-1-4939-8775-7_9	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85056393633&doi=10.1007%2f978-1-4939-8775-7_9&partnerID=40&md5=fe687e204c9738dd27757e8be536c4e5
199	Hosseini, M. and Pratas, D. and Pinho, A.J.	Cryfa: A secure encryption tool for genomic data	10.1093/bioinformatics/bty645	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85058744156&doi=10.1093%2fbioinformatics%2fbty645&partnerID=40&md5=41772737cf6b084637d2f679f8a92a97
202	Florea, A.G. and Buiu, C.	Using a web-based framework to build and experiment with virtual reality worlds	10.12753/2066-026X-19-037	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85085200368&doi=10.12753%2f2066-026X-19-037&partnerID=40&md5=4e303327c6127bdb86da73854ae6f6db
203	Naeem, M.R. and Lin, T. and Naeem, H. and Ullah, F. and Saeed, S.	Scalable Mutation Testing Using Predictive Analysis of Deep Learning Model	10.1109/ACCESS.2019.2950171	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85077961490&doi=10.1109%2fACCESS.2019.2950171&partnerID=40&md5=27b2b35bb352c07c7001b81e2adf5036
205	Garnier, R.	Dual vibration configuration interaction (DVCI). An efficient factorization of molecular Hamiltonian for high performance infrared spectrum computation	10.1016/j.cpc.2018.07.008	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85050756499&doi=10.1016%2fj.cpc.2018.07.008&partnerID=40&md5=6e272a47db8526585e8c79e4cfbe472f
206	Wei, Z.-G. and Zhang, S.-W.	DMSC: A dynamic multi-seeds method for clustering 16S rRNA sequences into OTUs	10.3389/fmicb.2019.00428	2019		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85066503067&doi=10.3389%2ffmicb.2019.00428&partnerID=40&md5=d9f525f0808d0f7a5fda680818768221
207	Siam, M. and Eikerdawy, S. and Gamal, M. and Abdel-Razek, M. and Jagersand, M. and Zhang, H.	Real-Time Segmentation with Appearance, Motion and Geometry	10.1109/IROS.2018.8594088	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85063000902&doi=10.1109%2fIROS.2018.8594088&partnerID=40&md5=34ea6c1b00f415c5a3f40b56db7fbb34
208	Marcos, D. and Tuia, D. and Kellenberger, B. and Zhang, L. and Bai, M. and Liao, R. and Urtasun, R.	Learning Deep Structured Active Contours End-to-End	10.1109/CVPR.2018.00925	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85057620146&doi=10.1109%2fCVPR.2018.00925&partnerID=40&md5=56d081f365989fcf85345d81139b5fb2
209	Carlson, J. and Li, J.Z. and Zöllner, S.	Helmsman: Fast and efficient mutation signature analysis for massive sequencing datasets	10.1186/s12864-018-5264-y	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85057527291&doi=10.1186%2fs12864-018-5264-y&partnerID=40&md5=2f37f0628dcc3910226a3b0c7c592d28
210	Yu, T. and Wang, T.	A Study of Regression Test Selection in Continuous Integration Environments	10.1109/ISSRE.2018.00024	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85059631520&doi=10.1109%2fISSRE.2018.00024&partnerID=40&md5=2f27a47720aac43fc231613f61ca6e54
211	Jiarpakdee, J. and Tantithamthavorn, C. and Treude, C.	Artefact: An r implementation of the autospearman function	10.1109/ICSME.2018.00083	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85058314837&doi=10.1109%2fICSME.2018.00083&partnerID=40&md5=bbfb7192701c5e999da7be8c1694a932
212	Kochhar, P.S. and Swierc, S. and Carnahan, T. and Sajnani, H. and Nagappan, M.	Understanding the role of reporting in work item tracking systems for software development: An industrial case study	10.1109/ICSME.2018.00070	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85058291545&doi=10.1109%2fICSME.2018.00070&partnerID=40&md5=ffc200d3d2fbc0766314f2b1772fb7d9
213	Thorve, S. and Sreshtha, C. and Meng, N.	An empirical study of flaky tests in android apps	10.1109/ICSME.2018.00062	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85058316108&doi=10.1109%2fICSME.2018.00062&partnerID=40&md5=fca6e18e7bb39ef9614f1b1eca187013
214	Saied, M.A. and Ouni, A. and Sahraoui, H. and Kula, R.G. and Inoue, K. and Lo, D.	Improving reusability of software libraries through usage pattern mining	10.1016/j.jss.2018.08.032	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85052331660&doi=10.1016%2fj.jss.2018.08.032&partnerID=40&md5=7c0289c5437875dae5fce4b41e59df2a
215	Jadoul, Y. and Thompson, B. and de Boer, B.	Introducing Parselmouth: A Python interface to Praat	10.1016/j.wocn.2018.07.001	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85050463066&doi=10.1016%2fj.wocn.2018.07.001&partnerID=40&md5=03eb1002bb7f4b3979c9e2c2c8658c92
216	Wessel, M. and De Souza, B.M. and Steinmacher, I. and Wiese, I.S. and Polato, I. and Chaves, A.P. and Gerosa, M.A.	The power of bots: Understanding bots in OSS projects	10.1145/3274451	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85064157845&doi=10.1145%2f3274451&partnerID=40&md5=9b136fc43db8f92bbd51f215980b01ba
217	Shelenkov, A.A. and Slavokhotova, A.A. and Odintsova, T.I.	Cysmotif Searcher Pipeline for Antimicrobial Peptide Identification in Plant Transcriptomes	10.1134/S0006297918110135	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85056605620&doi=10.1134%2fS0006297918110135&partnerID=40&md5=07792323d67216973fab744cbf2d32c7
218	Reinhardt, A. and Zhang, T. and Mathur, M. and Kim, M.	Augmenting stack overflow with api usage patterns mined from GitHub	10.1145/3236024.3264585	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85058345090&doi=10.1145%2f3236024.3264585&partnerID=40&md5=e8397a485a95e724c3d27c0d7ea978f2
219	Basios, M. and Li, L. and Wu, F. and Kanthan, L. and Barr, E.T.	Darwinian data structure selection	10.1145/3236024.3236043	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85058345723&doi=10.1145%2f3236024.3236043&partnerID=40&md5=5ef469be25b68e9331a23f8b1f799281
220	Lipčák, J. and Rossi, B.	A large-scale study on source code reviewer recommendation	10.1109/SEAA.2018.00068	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85057204726&doi=10.1109%2fSEAA.2018.00068&partnerID=40&md5=525ca30e887f1c7fac89306318ee8e08
221	Kómár, P. and Kural, D.	Geck: Trio-based comparative benchmarking of variant calls	10.1093/bioinformatics/bty415	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85054889812&doi=10.1093%2fbioinformatics%2fbty415&partnerID=40&md5=507b60687c8cad2c25a289fe254afdfd
223	Rácz, A. and Bajusz, D. and Héberger, K.	Life beyond the Tanimoto coefficient: Similarity measures for interaction fingerprints	10.1186/s13321-018-0302-y	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85054537670&doi=10.1186%2fs13321-018-0302-y&partnerID=40&md5=a7a9a49e18b5c6f0441baac5069016ac
224	Wang, H. and Marcišauskas, S. and Sánchez, B.J. and Domenzain, I. and Hermansson, D. and Agren, R. and Nielsen, J. and Kerkhoven, E.J.	RAVEN 2.0: A versatile toolbox for metabolic network reconstruction and a case study on Streptomyces coelicolor	10.1371/journal.pcbi.1006541	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85055635316&doi=10.1371%2fjournal.pcbi.1006541&partnerID=40&md5=8e34d8ac36ffec399a40f59cbd2b622d
225	Saborido, R. and Morales, R. and Khomh, F. and Guéhéneuc, Y.-G. and Antoniol, G.	Getting the most from map data structures in Android	10.1007/s10664-018-9607-8	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85045030669&doi=10.1007%2fs10664-018-9607-8&partnerID=40&md5=5f54b44ea67e32903abf4b76bf2ec290
226	Moore, F.C. and Rising, J. and Lollo, N. and Springer, C. and Vasquez, V. and Dolginow, A. and Hope, C. and Anthoff, D.	Mimi-PAGE, an open-source implementation of the PAGE09 integrated assessment model	10.1038/sdata.2018.187	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85054070490&doi=10.1038%2fsdata.2018.187&partnerID=40&md5=0814b96e4538cc1c69fe57deced69bd9
227	Ahmar, Y.E. and Pallec, X.L. and Gérard, S.	The visual variables in UML: How are they used by women?	10.1145/3241403.3241422	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85055689848&doi=10.1145%2f3241403.3241422&partnerID=40&md5=01113c866ec9636a87b7f378d24abf4b
228	Rosenfeld, A.M. and Meng, W. and Luning Prak, E.T. and Hershberg, U.	ImmuneDB, a novel tool for the analysis, storage, and dissemination of immune repertoire sequencing data	10.3389/fimmu.2018.02107	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85054779808&doi=10.3389%2ffimmu.2018.02107&partnerID=40&md5=fa6a1c028dde9cd542ec559fcdf5d7eb
229	Janes, A. and Mairegger, M. and Russo, B.	Code_Call_lens: Raising the developer awareness of critical code	10.1145/3238147.3240488	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85056520236&doi=10.1145%2f3238147.3240488&partnerID=40&md5=5bf69491015f1e6896e7ffb68bc4a1f2
230	Pereira, R. and Cunha, J. and Simão, P. and Saraiva, J.	JStanley: Placing a green thumb on Java collections	10.1145/3238147.3240473	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85054401366&doi=10.1145%2f3238147.3240473&partnerID=40&md5=0051344416edf38f1aac15d8b6af79c0
231	Ankenbrand, M.J. and Hohlfeld, S.C.Y. and Weber, L. and Förster, F. and Keller, A.	Functional exploration of natural networks and ecological communities	10.1111/2041-210X.13060	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85052824894&doi=10.1111%2f2041-210X.13060&partnerID=40&md5=7e00e124ca08f173cfbe92d5db40b63d
232	Yang, D.	SolarData: An R package for easy access of publicly available solar datasets	10.1016/j.solener.2018.06.107	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85052434812&doi=10.1016%2fj.solener.2018.06.107&partnerID=40&md5=ebca0e528fc16b539bd612e5648ddddb
233	Jain, C. and Koren, S. and Dilthey, A. and Phillippy, A.M. and Aluru, S.	A fast adaptive algorithm for computing whole-genome homology maps	10.1093/bioinformatics/bty597	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85054146576&doi=10.1093%2fbioinformatics%2fbty597&partnerID=40&md5=67c62cff95e80cebc214a79014e00a86
234	Behera, S. and Gayen, S. and Deogun, J.S. and Vinodchandran, N.V.	KmerEstimate: A Streaming Algorithm for Estimating k-mer Counts with Optimal Space Usage	10.1145/3233547.3233587	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85056109624&doi=10.1145%2f3233547.3233587&partnerID=40&md5=51a14ae8ffe7a46048ae942c44dbf6af
235	Borle, N.C. and Feghhi, M. and Stroulia, E. and Greiner, R. and Hindle, A.	Analyzing the effects of test driven development in GitHub	10.1007/s10664-017-9576-3	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85034827767&doi=10.1007%2fs10664-017-9576-3&partnerID=40&md5=694f065a257ee7566d89600b6d2db3a7
236	Sawant, A.A. and Robbes, R. and Bacchelli, A.	On the reaction to deprecation of clients of 4 + 1 popular Java APIs and the JDK	10.1007/s10664-017-9554-9	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85031416657&doi=10.1007%2fs10664-017-9554-9&partnerID=40&md5=00e478fbaaf19937c9e60b62e5bd0d5d
237	Echegaray, S. and Bakr, S. and Rubin, D.L. and Napel, S.	Quantitative Image Feature Engine (QIFE): an Open-Source, Modular Engine for 3D Quantitative Feature Extraction from Volumetric Medical Images	10.1007/s10278-017-0019-x	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85030866982&doi=10.1007%2fs10278-017-0019-x&partnerID=40&md5=43a61493b60214673086ca2e440a145b
238	Tian, Y. and Wang, X. and Sun, H. and Zhao, Y. and Guo, C. and Liu, X.	Automatically Generating API Usage Patterns from Natural Language Queries	10.1109/APSEC.2018.00020	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85066793048&doi=10.1109%2fAPSEC.2018.00020&partnerID=40&md5=0623756ba3dd5369cd7480225b72a40d
239	Nguyen, T. and Chen, S.W. and Shivakumar, S.S. and Taylor, C.J. and Kumar, V.	Unsupervised Deep Homography: A Fast and Robust Homography Estimation Model	10.1109/LRA.2018.2809549	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85052248455&doi=10.1109%2fLRA.2018.2809549&partnerID=40&md5=f656755a68b252433ed58e7643541144
240	Dube, R. and Gollub, M.G. and Sommer, H. and Gilitschenski, I. and Siegwart, R. and Cadena, C. and Nieto, J.	Incremental-segment-based localization in 3-d point clouds	10.1109/LRA.2018.2803213	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85053825306&doi=10.1109%2fLRA.2018.2803213&partnerID=40&md5=84ae30043a7de46d002bb818eba51185
241	Mak, L. and Li, M. and Cao, C. and Gordon, P. and Tarailo-Graovac, M. and Bousman, C. and Wang, P. and Long, Q.	SimPEL: Simulation-based power estimation for sequencing studies of low-prevalence conditions	10.1002/gepi.22129	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85049050190&doi=10.1002%2fgepi.22129&partnerID=40&md5=0beb355dccad65c5d4eed055ae59feac
242	Król, M. and Reñé, S. and Ascigil, O. and Psaras, I.	ChainSoft: Collaborative software development using smart contracts	10.1145/3211933.3211934	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85051677658&doi=10.1145%2f3211933.3211934&partnerID=40&md5=828ea74bb8f6e6b411ab0842f37dd8bc
243	Sun, X. and Fu, Y. and Dong, Y. and Liu, Z. and Zhang, Y.	Improving Fitness Function for Language Fuzzing with PCFG Model	10.1109/COMPSAC.2018.00098	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85055429872&doi=10.1109%2fCOMPSAC.2018.00098&partnerID=40&md5=04693d144f3e126836ccc3998ae9811a
244	Barletta, A. and de Magistris, P.S.	Analyzing the risks embedded in option prices with rndfittool	10.3390/risks6020028	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85056743634&doi=10.3390%2frisks6020028&partnerID=40&md5=51b2c4bdd8a5c709ac11087c5b3595dc
245	Laurin-Lemay, S. and Philippe, H. and Rodrigue, N.	Multiple factors confounding phylogenetic detection of selection on codon usage	10.1093/molbev/msy047	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85059961771&doi=10.1093%2fmolbev%2fmsy047&partnerID=40&md5=66ecba6335f0448b2a789e0c132e70c8
246	Sun, W. and Duan, T. and Ye, P. and Chen, K. and Zhang, G. and Lai, M. and Zhang, H.	TSVdb: A web-tool for TCGA splicing variants analysis	10.1186/s12864-018-4775-x	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85047824127&doi=10.1186%2fs12864-018-4775-x&partnerID=40&md5=d61a3fc555c265b7114a28f8e59eead7
247	Porkoláb, Z. and Brunner, T. and Krupp, D. and Csordás, M.	Codecompass: An open software comprehension framework for industrial usage	10.1145/3196321.3197546	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85051672172&doi=10.1145%2f3196321.3197546&partnerID=40&md5=73c6d2642b37cd747c16c9dcc269d663
249	Porkoláb, Z. and Brunner, T.	The codecompass comprehension framework	10.1145/3196321.3196352	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85051663783&doi=10.1145%2f3196321.3196352&partnerID=40&md5=bc1df16b024d36c28838384a22de85dc
250	Zhang, T. and Upadhyaya, G. and Reinhardt, A. and Rajan, H. and Kim, M.	Are code examples on an online Q&amp;A forum reliable?: A study of API misuse on stack overflow	10.1145/3180155.3180260	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85046979029&doi=10.1145%2f3180155.3180260&partnerID=40&md5=c6a58327c8bd3e25e13ca2b8a213878e
251	Kochhar, P.S. and Swierc, S. and Carnahan, T. and Sajnani, H. and Nagappan, M.	Poster: Understanding the role of reporting in work item tracking systems for software development: An industrial case study	10.1145/3183440.3195071	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85049697318&doi=10.1145%2f3183440.3195071&partnerID=40&md5=2a79c5fd1372850135457b0f56e05a35
252	Banerjee, A. and Chong, L.K. and Ballabriga, C. and Roychoudhury, A.	EnergyPatch: Repairing Resource Leaks to Improve Energy-Efficiency of Android Apps	10.1109/TSE.2017.2689012	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85047012040&doi=10.1109%2fTSE.2017.2689012&partnerID=40&md5=90975c442961d54a3871d02bc1433323
253	Hamouda, E. and Sara, E.-M. and Tarek, M.	Ant lion optimization algorithm for kidney exchanges	10.1371/journal.pone.0196707	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85046684400&doi=10.1371%2fjournal.pone.0196707&partnerID=40&md5=a8d4be9994b998b5e4425756ad3ad451
254	Kolomiiets, B. and Popov, A.	System for registration and analysis of human stabilograms	10.1109/TCSET.2018.8336288	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85047462467&doi=10.1109%2fTCSET.2018.8336288&partnerID=40&md5=ebefd9e8903f82cfb80a066cb2c2dba1
255	Yang, S. and Wang, J. and Ng, R.T.	Inferring RNA sequence preferences for poorly studied RNA-binding proteins based on co-evolution	10.1186/s12859-018-2091-8	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85043479956&doi=10.1186%2fs12859-018-2091-8&partnerID=40&md5=9b6c603696039e7d46ca46ef0e6319ee
256	Pikelner, A.	FMFT: fully massive four-loop tadpoles	10.1016/j.cpc.2017.11.017	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85039705695&doi=10.1016%2fj.cpc.2017.11.017&partnerID=40&md5=a773ddff82eb65a0cd8cc7662be6caf5
257	Sweeney, P.W. and Starly, B. and Morris, P.J. and Xu, Y. and Jones, A. and Radhakrishnan, S. and Grassa, C.J. and Davis, C.C.	Large-scale digitization of herbarium specimens: Development and usage of an automated, high-throughput conveyor system	10.12705/671.9	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85043323449&doi=10.12705%2f671.9&partnerID=40&md5=6d1c3d1a07a13e0a37dc5f5dc034ea1f
258	Madanecki, P. and Bałut, M. and Buckley, P.G. and Renata Ochocka, J. and Bartoszewski, R. and Crossman, D.K. and Messiaen, L.M. and Piotrowski, A.	High-Throughput Tabular Data Processor – Platform independent graphical tool for processing large data sets	10.1371/journal.pone.0192858	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85042121876&doi=10.1371%2fjournal.pone.0192858&partnerID=40&md5=97797d98be9811ba45bc4e026a6c3d98
259	Zampieri, G. and Tran, D.V. and Donini, M. and Navarin, N. and Aiolli, F. and Sperduti, A. and Valle, G.	Scuba: Scalable kernel-based gene prioritization	10.1186/s12859-018-2025-5	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85041449628&doi=10.1186%2fs12859-018-2025-5&partnerID=40&md5=01edcf5d46e4535cb773534048284e60
260	Probst, D. and Reymond, J.-L.	SmilesDrawer: Parsing and Drawing SMILES-Encoded Molecular Structures Using Client-Side JavaScript	10.1021/acs.jcim.7b00425	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85040905901&doi=10.1021%2facs.jcim.7b00425&partnerID=40&md5=4717030ff906eb388bffacef5db14f11
261	Diaz-del-Pino, S. and Trelles, O. and Falgueras, J.	mORCA: Ubiquitous access to life science web services	10.1186/s12864-018-4439-x	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85040740007&doi=10.1186%2fs12864-018-4439-x&partnerID=40&md5=76f83b9ae8787584fc14bd69337ce090
262	Rao, D. and Dubey, S. and Mohan Kumar, J. and Rao, D. and Balaji, B.	Descriptive and distribution analysis of GitHub repository data	10.14419/ijet.v7i3.12489	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85077664193&doi=10.14419%2fijet.v7i3.12489&partnerID=40&md5=6b7a1840e3ef8dbe38969a031b8d8c9b
263	Härtel, J. and Heinz, M. and Lämmel, R.	EMF patterns of usage on github	10.1007/978-3-319-92997-2_14	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85048775242&doi=10.1007%2f978-3-319-92997-2_14&partnerID=40&md5=ee3d49a87e988ab0c914ad0c07fb6859
264	Laurin-Lemay, S. and Rodrigue, N. and Lartillot, N. and Philippe, H.	Conditional approximate Bayesian computation: A new approach for across-site dependency in high-dimensional mutation-selection models	10.1093/molbev/msy173	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85056310223&doi=10.1093%2fmolbev%2fmsy173&partnerID=40&md5=3a96d6e48c7c744e6d755de4e73a3952
265	Kohl, T.A. and Utpatel, C. and Schleusener, V. and De Filippo, M.R. and Beckert, P. and Cirillo, D.M. and Niemann, S.	MTBseq: A comprehensive pipeline for whole genome sequence analysis of Mycobacterium tuberculosis complex isolates	10.7717/peerj.5895	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85054851920&doi=10.7717%2fpeerj.5895&partnerID=40&md5=67f6e2f61b0f0be1d76ba9a4908dda00
266	Chaudhary, H.A.A. and Hassan, S.-U. and Aljohani, N.R. and Daud, A.	Quality classification of scientific publications using hybrid summarization model	10.1007/978-3-030-04257-8_6	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85057262907&doi=10.1007%2f978-3-030-04257-8_6&partnerID=40&md5=9ab2c4aa9f76475bd2d23c4473c268bb
267	Oliveira, J. and Viggiato, M. and Santos, M. and Figueiredo, E. and Marques-Neto, H.	An empirical study on the impact of android code smells on resource usage	10.18293/SEKE2018-157	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85056839301&doi=10.18293%2fSEKE2018-157&partnerID=40&md5=8822f5556d206c977080599d910c09a5
270	Raja, M.A. and Ryan, C.	GELAB - A Matlab toolbox for grammatical evolution	10.1007/978-3-030-03496-2_22	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85057089290&doi=10.1007%2f978-3-030-03496-2_22&partnerID=40&md5=5cf886665a07789453b78c5803363235
271	Li, Y. and Ouyang, W. and Zhou, B. and Shi, J. and Zhang, C. and Wang, X.	Factorizable Net: An Efficient Subgraph-Based Framework for Scene Graph Generation	10.1007/978-3-030-01246-5_21	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85055105093&doi=10.1007%2f978-3-030-01246-5_21&partnerID=40&md5=82cb20743c257f2f2af0dd3329566ded
272	Barido-Sottani, J. and Bošková, V. and Plessis, L.D. and Kühnert, D. and Magnus, C. and Mitov, V. and Müller, N.F. and Pečerska, J. and Rasmussen, D.A. and Zhang, C. and Drummond, A.J. and Heath, T.A. and Pybus, O.G. and Vaughan, T.G. and Stadler, T.	Taming the BEAST - A Community Teaching Material Resource for BEAST 2	10.1093/sysbio/syx060	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85040108507&doi=10.1093%2fsysbio%2fsyx060&partnerID=40&md5=44ab357256d08be7626ffbdb7da30fd9
273	Sethi, T. and Maheshwari, S. and Nagori, A. and Lodha, R.	Stewarding antibiotic stewardship in intensive care units with bayesian artificial intelligence [version 1; peer review: 2 approved with reservations]	10.12688/wellcomeopenres.14629.1	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85078336824&doi=10.12688%2fwellcomeopenres.14629.1&partnerID=40&md5=df738b35c6342dc93955adf281e1687a
274	Endara, L. and Thessen, A.E. and Cole, H.A. and Walls, R. and Gkoutos, G. and Cao, Y. and Chong, S.S. and Cui, H.	Modifier Ontologies for frequency, certainty, degree, and coverage phenotype modifier	10.3897/BDJ.6.E29232	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85082880742&doi=10.3897%2fBDJ.6.E29232&partnerID=40&md5=fe451943a15591a55547c7ca23c0c143
275	Borowka, S. and Heinrich, G. and Jahn, S. and Jones, S.P. and Kerner, M. and Schlenk, J. and Zirke, T.	pySecDec: A toolbox for the numerical evaluation of multi-scale integrals	10.1016/j.cpc.2017.09.015	2018		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85031431867&doi=10.1016%2fj.cpc.2017.09.015&partnerID=40&md5=2805a13dcd12f1980f7a7aa9487bf3b5
276	Hou, S. and Feng, Y. and Wang, Z.	VegFru: A Domain-Specific Dataset for Fine-Grained Visual Categorization	10.1109/ICCV.2017.66	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85041895074&doi=10.1109%2fICCV.2017.66&partnerID=40&md5=16516f9dc358a4b8cd0dd4b16d6e3c8c
277	Kafkas, Ş. and Sarntivijai, S. and Hoehndorf, R.	Usage of cell nomenclature in biomedical literature	10.1186/s12859-017-1978-0	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85038955885&doi=10.1186%2fs12859-017-1978-0&partnerID=40&md5=9b68bd9d36bf5a42c09bc7900af8fd53
278	Costin, A.	Lua Code: Security overview and practical approaches to static analysis	10.1109/SPW.2017.38	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85048319697&doi=10.1109%2fSPW.2017.38&partnerID=40&md5=c6a34231af4721b9c24605fe4ae8c15b
279	Poehlman, W.L. and Rynge, M. and Balamurugan, D. and Mills, N. and Feltus, F.A.	OSG-KINC: High-throughput gene co-expression network construction using the open science grid	10.1109/BIBM.2017.8217938	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85046091228&doi=10.1109%2fBIBM.2017.8217938&partnerID=40&md5=beb2f916034fcaad283d377cf699193c
280	Chapman, C. and Wang, P. and Stolee, K.T.	Exploring regular expression comprehension	10.1109/ASE.2017.8115653	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85041430882&doi=10.1109%2fASE.2017.8115653&partnerID=40&md5=0c8a512b78ca4c961eb8c540762d12f1
281	Alimoglu, A. and Ozturan, C.	Design of a smart contract based autonomous organization for sustainable software	10.1109/eScience.2017.76	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85043760581&doi=10.1109%2feScience.2017.76&partnerID=40&md5=b76e002bff8bb638602886737835f3e3
282	Xu, W. and Sun, X. and Hu, J. and Li, B.	REPERSP: Recommending personalized software projects on GitHub	10.1109/ICSME.2017.20	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85032496394&doi=10.1109%2fICSME.2017.20&partnerID=40&md5=72ddb256bacfb19352d83e4dab953357
283	Herz, C. and Fillion-Robin, J.-C. and Onken, M. and Riesmeier, J. and Lasso, A. and Pinter, C. and Fichtinger, G. and Pieper, S. and Clunie, D. and Kikinis, R. and Fedorov, A.	dcmqi: An open source library for standardized communication of quantitative image analysis results using DICOM	10.1158/0008-5472.CAN-17-0336	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85033433885&doi=10.1158%2f0008-5472.CAN-17-0336&partnerID=40&md5=bf0635dc9fe3c75ac5aab22569514e56
284	Sayagh, M. and Dong, Z. and Andrzejak, A. and Adams, B.	Does the Choice of Configuration Framework Matter for Developers? Empirical Study on 11 Java Configuration Frameworks	10.1109/SCAM.2017.25	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85047197551&doi=10.1109%2fSCAM.2017.25&partnerID=40&md5=dfd9358cd85fb3cfb66ee6bbd6678f22
285	Duan, R. and Bijlani, A. and Xu, M. and Kim, T. and Lee, W.	Identifying open-source license violation and 1-day security risk at large scale	10.1145/3133956.3134048	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85041441238&doi=10.1145%2f3133956.3134048&partnerID=40&md5=495b5e41caa6c4796f021dcf554a9ecf
286	Stacey, R.G. and Skinnider, M.A. and Scott, N.E. and Foster, L.J.	A rapid and accurate approach for prediction of interactomes from co-elution data (PrInCE)	10.1186/s12859-017-1865-8	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85031900758&doi=10.1186%2fs12859-017-1865-8&partnerID=40&md5=673cb14e9f6fa15a77ecee9f0e9005d8
287	Hosseinabady, M. and Nunez-Yanez, J.L.	A systematic approach to design and optimise streaming applications on FPGA using high-level synthesis	10.23919/FPL.2017.8056758	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85034445484&doi=10.23919%2fFPL.2017.8056758&partnerID=40&md5=9ab4f54a0b6f8eb12d01877a154b339d
288	Verdegem, D. and Moseley, H.N.B. and Vermaelen, W. and Sanchez, A.A. and Ghesquière, B.	MAIMS: a software tool for sensitive metabolic tracer analysis through the deconvolution of 13C mass isotopologue profiles of large composite metabolites	10.1007/s11306-017-1250-7	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85029167235&doi=10.1007%2fs11306-017-1250-7&partnerID=40&md5=81430f39c1aae5a491233232e73145c3
291	Limasset, A. and Rizk, G. and Chikhi, R. and Peterlongo, P.	Fast and scalable minimal perfect hashing for massive key sets	10.4230/LIPIcs.SEA.2017.25	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85028774755&doi=10.4230%2fLIPIcs.SEA.2017.25&partnerID=40&md5=94a707c55201e871d0968c7a877b7dd0
292	Silva, L.H. and Valente, M.T. and Bergel, A. and Anquetil, N. and Etien, A.	Identifying Classes in Legacy JavaScript Code	10.1002/smr.1864	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85026836053&doi=10.1002%2fsmr.1864&partnerID=40&md5=e3ca1bd5a022845d4038e4dd9fa7474e
293	Almodaresi, F. and Pandey, P. and Patro, R.	Rainbowfish: A succinct colored de Bruijn graph representation	10.4230/LIPIcs.WABI.2017.18	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85028766603&doi=10.4230%2fLIPIcs.WABI.2017.18&partnerID=40&md5=875e4b2c4f555353948db4252094bd3f
294	Yang, D. and Tian, D. and Gong, J. and Gao, S. and Yang, T. and Li, X.	Difference Bloom Filter: A probabilistic structure for multi-set membership query	10.1109/ICC.2017.7996678	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85028305891&doi=10.1109%2fICC.2017.7996678&partnerID=40&md5=69043350670573ce88e28c1865bff34e
295	Wei, Z.-G. and Zhang, S.-W.	DBH: A de Bruijn graph-based heuristic method for clustering large-scale 16S rRNA sequences into OTUs	10.1016/j.jtbi.2017.04.019	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85019417482&doi=10.1016%2fj.jtbi.2017.04.019&partnerID=40&md5=f7d17cb21ee1767941e4f460de24e7d9
296	Schröder, J. and Wirawan, A. and Schmidt, B. and Papenfuss, A.T.	CLOVE: Classification of genomic fusions into structural variation events	10.1186/s12859-017-1760-3	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85025113789&doi=10.1186%2fs12859-017-1760-3&partnerID=40&md5=f93c82f6be3cf66b51779292d851b01c
297	Khatchadourian, R. and Masuhara, H.	Automated Refactoring of Legacy Java Software to Default Methods	10.1109/ICSE.2017.16	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85027716666&doi=10.1109%2fICSE.2017.16&partnerID=40&md5=1e70ebaaf5b72fe1b6621011225a9fcb
298	Khelik, K. and Lagesen, K. and Sandve, G.K. and Rognes, T. and Nederbragt, A.J.	NucDiff: In-depth characterization and annotation of differences between two sets of DNA sequences	10.1186/s12859-017-1748-z	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85023207388&doi=10.1186%2fs12859-017-1748-z&partnerID=40&md5=d5aaababc8605f99cee76099bc2a4dc4
299	Walsh, T.A. and Kapfhammer, G.M. and McMinn, P.	ReDeCheck: An automatic layout failure checking tool for responsively designed web pages	10.1145/3092703.3098221	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85026667206&doi=10.1145%2f3092703.3098221&partnerID=40&md5=dbc20af7124e7187a303b6992df11ff2
300	Dennis, C. and Krutz, D.E. and Wiem Mkaouer, M.	P-Lint: A Permission Smell Detector for Android Applications	10.1109/MOBILESoft.2017.24	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85027037638&doi=10.1109%2fMOBILESoft.2017.24&partnerID=40&md5=d36faf195f044547d79802a4c659f3ef
301	Teleman, J. and Hauri, S. and Malmström, J.	Improvements in Mass Spectrometry Assay Library Generation for Targeted Proteomics	10.1021/acs.jproteome.6b00928	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85023172993&doi=10.1021%2facs.jproteome.6b00928&partnerID=40&md5=feba444e0c4f33b35a3eafd928397b58
302	Zhou, Y. and Liu, P. and Jin, H. and Yang, T. and Dang, S. and Li, X.	One Memory Access Sketch: A More Accurate and Faster Sketch for Per-Flow Measurement	10.1109/GLOCOM.2017.8254741	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85046397838&doi=10.1109%2fGLOCOM.2017.8254741&partnerID=40&md5=10209a7a9eb1a6aa338692386deced4d
303	Yang, T. and Yin, B. and Li, H. and Shahzad, M. and Uhlig, S. and Cm, B. and Li, X.	Rectangular hash table: Bloom filter and bitmap assisted hash table with high speed	10.1109/BigData.2017.8257999	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85047736321&doi=10.1109%2fBigData.2017.8257999&partnerID=40&md5=f90176a81889548fe771fc1e845f27ec
304	Baltes, S. and Kiefer, R. and Diehl, S.	Attribution required: Stack overflow code snippets in GitHub projects	10.1109/ICSE-C.2017.99	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85026730072&doi=10.1109%2fICSE-C.2017.99&partnerID=40&md5=f150a26c7ace22743bfe213b6992e49e
305	Zampetti, F. and Scalabrino, S. and Oliveto, R. and Canfora, G. and Di Penta, M.	How Open Source Projects Use Static Code Analysis Tools in Continuous Integration Pipelines	10.1109/MSR.2017.2	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85026532196&doi=10.1109%2fMSR.2017.2&partnerID=40&md5=faeb88b4c5b6ea11a5a7438cdbd37396
306	Malaquias, R. and Ribeiro, M. and Bonifacio, R. and Monteiro, E. and Medeiros, F. and Garcia, A. and Gheyi, R.	The Discipline of Preprocessor-Based Annotations - Does #ifdef TAG n't #endif Matter	10.1109/ICPC.2017.41	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85025173309&doi=10.1109%2fICPC.2017.41&partnerID=40&md5=a9777caa317869cded2555dbcf790122
308	Chen, X. and Wang, C. and Tang, S. and Yu, C. and Zou, Q.	CMSA: A heterogeneous CPU/GPU computing system for multiple similar RNA/DNA sequence alignment	10.1186/s12859-017-1725-6	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85021212447&doi=10.1186%2fs12859-017-1725-6&partnerID=40&md5=2af6ffd652936e4b554aa0a5b7ed3a0e
309	Kochhar, P.S. and Lo, D.	Revisiting assert use in GitHub projects	10.1145/3084226.3084259	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85025455377&doi=10.1145%2f3084226.3084259&partnerID=40&md5=189313fa545da7d77ed93ef6231410dc
310	Sawant, A.A. and Bacchelli, A.	fine-GRAPE: fine-grained APi usage extractor – an approach and dataset to investigate API usage	10.1007/s10664-016-9444-6	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84982947678&doi=10.1007%2fs10664-016-9444-6&partnerID=40&md5=8f44f4b75d284872cdcf6a7d8ad72b83
311	Vendome, C. and Bavota, G. and Penta, M.D. and Linares-Vásquez, M. and German, D. and Poshyvanyk, D.	License usage and changes: a large-scale study on gitHub	10.1007/s10664-016-9438-4	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85020033576&doi=10.1007%2fs10664-016-9438-4&partnerID=40&md5=8854e37b29ad157bd8c1251cb52edb99
312	Cardall, C.Y. and Budiardja, R.D.	GENASIS   Basics: Object-oriented utilitarian functionality for large-scale physics simulations (Version 2)	10.1016/j.cpc.2016.12.019	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85011898489&doi=10.1016%2fj.cpc.2016.12.019&partnerID=40&md5=79804a4975e39e2c6ef3fe81bfd064fa
313	Beshero-Bondar, E.E. and Parker, R.J.	A GitHub garage for a digital humanities course	10.1007/978-3-319-54226-3_15	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85033979673&doi=10.1007%2f978-3-319-54226-3_15&partnerID=40&md5=b998a0c04dd71e6e844e40438dbe142a
314	Leitner, P. and Bezemer, C.-P.	An exploratory study of the state of practice of performance testing in Java-based open source projects	10.1145/3030207.3030213	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85019046338&doi=10.1145%2f3030207.3030213&partnerID=40&md5=014e9d6263420a8166cf42cbaa28c2ff
315	Edmands, W.M.B. and Petrick, L. and Barupal, D.K. and Scalbert, A. and Wilson, M.J. and Wickliffe, J.K. and Rappaport, S.M.	CompMS2Miner: An Automatable Metabolite Identification, Visualization, and Data-Sharing R Package for High-Resolution LC-MS Data Sets	10.1021/acs.analchem.6b02394	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85019765844&doi=10.1021%2facs.analchem.6b02394&partnerID=40&md5=b1c4f0a196637eadeff88fbe89dc6928
317	Zhang, Y. and Lo, D. and Kochhar, P.S. and Xia, X. and Li, Q. and Sun, J.	Detecting similar repositories on GitHub	10.1109/SANER.2017.7884605	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85018437178&doi=10.1109%2fSANER.2017.7884605&partnerID=40&md5=f997ac4638de3de022c5a14bd3b52d2e
318	Zerouali, A. and Mens, T.	Analyzing the evolution of testing library usage in open source Java projects	10.1109/SANER.2017.7884645	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85018411983&doi=10.1109%2fSANER.2017.7884645&partnerID=40&md5=2087e3cb3f3941d46fb4331c158e8d6a
319	Ouni, A. and Kula, R.G. and Kessentini, M. and Ishio, T. and German, D.M. and Inoue, K.	Search-based software library recommendation using multi-objective optimization	10.1016/j.infsof.2016.11.007	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85006282719&doi=10.1016%2fj.infsof.2016.11.007&partnerID=40&md5=92b41cc2c123052c8bbf93cd91b59a6a
320	Sandi, S. and Krstajic, B. and Popovic, T.	PyPMU - Open source python package for synchrophasor data transfer	10.1109/TELFOR.2016.7818916	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85013665825&doi=10.1109%2fTELFOR.2016.7818916&partnerID=40&md5=3b262e34a23874997c4149d9869469b8
321	Sliwko, L. and Getov, V.	AGOCS - Accurate Google Cloud Simulator Framework	10.1109/UIC-ATC-ScalCom-CBDCom-IoP-SmartWorld.2016.0095	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85013219936&doi=10.1109%2fUIC-ATC-ScalCom-CBDCom-IoP-SmartWorld.2016.0095&partnerID=40&md5=184496035a674337e22edcb0da751920
322	Gaye, A.	Extending the R Library PROPER to enable power calculations for isoform-level analysis with EBSeq	10.3389/fgene.2016.00225	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85012202199&doi=10.3389%2ffgene.2016.00225&partnerID=40&md5=d64c6f12fca7588333124e35170075cf
323	Audain, E. and Uszkoreit, J. and Sachsenberg, T. and Pfeuffer, J. and Liang, X. and Hermjakob, H. and Sanchez, A. and Eisenacher, M. and Reinert, K. and Tabb, D.L. and Kohlbacher, O. and Perez-Riverol, Y.	In-depth analysis of protein inference algorithms using multiple search engines and well-defined metrics	10.1016/j.jprot.2016.08.002	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84988843652&doi=10.1016%2fj.jprot.2016.08.002&partnerID=40&md5=4d4b37efcc310c115168539d2dd38469
324	Roveda, R. and Fontana, F.A. and Raibulet, C. and Zanoni, M. and Rampazzo, F.	Does the migration to GitHub relate to internal software quality?	10.5220/0006367402930300	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85025468384&doi=10.5220%2f0006367402930300&partnerID=40&md5=8bc66fdfbec847b1b7b48970028b8129
325	Orlov, S.	C++ playground for numerical integration method developers	10.1007/978-3-319-71255-0_34	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85035098847&doi=10.1007%2f978-3-319-71255-0_34&partnerID=40&md5=d0d1685f20bb8c60ad8a2d5c089d899c
326	Lu, H. and Zhou, Y. and Zhang, Z.-K.	Calculate deep convolution NeurAl network on cell unit	10.1007/978-981-10-4154-9_61	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85016119646&doi=10.1007%2f978-981-10-4154-9_61&partnerID=40&md5=8d6f45d34a83f96eafec8664106fcceb
329	Bond, S.R. and Keat, K.E. and Barreira, S.N. and Baxevanis, A.D.	BuddySuite: Command-line toolkits for manipulating sequences, alignments, and phylogenetic trees	10.1093/molbev/msx089	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85027411555&doi=10.1093%2fmolbev%2fmsx089&partnerID=40&md5=ccf3f18cba3e7affdf75f6afd373e8dd
330	Santos, M.R. and de Sá, T.Q.V. and da Silva, F.E. and dos Santos Junior, M.R. and Maia, T.A. and Reis, Z.S.N.	Health information exchange for continuity of maternal and neonatal care supporting: A proof-of-concept based on ISO standard	10.4338/ACI-2017-06-RA-0106	2017		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85034568205&doi=10.4338%2fACI-2017-06-RA-0106&partnerID=40&md5=2fff9fb8b35315185d9f93f7e35be1c1
331	Feist, M.D. and Santos, E.A. and Watts, I. and Hindle, A.	Visualizing Project Evolution through Abstract Syntax Tree Analysis	10.1109/VISSOFT.2016.6	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85010424321&doi=10.1109%2fVISSOFT.2016.6&partnerID=40&md5=c8e471fc733147a53b49a2f5b195474d
332	Kim, D. and Hahn, A.S. and Hanson, N.W. and Konwar, K.M. and Hallam, S.J.	FAST: Fast annotation with synchronized threads	10.1109/CIBCB.2016.7758120	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85011392004&doi=10.1109%2fCIBCB.2016.7758120&partnerID=40&md5=d889a8bd6b390ecff7a28f67f715cfc2
333	Fowkes, J. and Sutton, C.	Parameter-Free Probabilistic API Mining across GitHub	10.1145/2950290.2950319	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84997219095&doi=10.1145%2f2950290.2950319&partnerID=40&md5=1e53b808fa56f7c08d3bd67cc57990af
334	Gu, X. and Zhang, H. and Zhang, D. and Kim, S.	Deep API Learning	10.1145/2950290.2950334	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84997108965&doi=10.1145%2f2950290.2950334&partnerID=40&md5=ae730cb574590851b20d956bb285b909
335	Toal, R. and Rivera, R. and Schneider, A. and Choe, E.	Programming Language Explorations	10.1201/b21848	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85042370650&doi=10.1201%2fb21848&partnerID=40&md5=c1e51c41e3ebffc4ceda1a810326c892
336	Shen, W. and Le, S. and Li, Y. and Hu, F.	SeqKit: A cross-platform and ultrafast toolkit for FASTA/Q file manipulation	10.1371/journal.pone.0163962	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84992166172&doi=10.1371%2fjournal.pone.0163962&partnerID=40&md5=150eafb99a3677230d571a0e7ca5dc3d
360	Eck, B.J.	EpanetReader: A Package for Reading EPANET Files into R	10.1061/9780784479865.051	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84976520090&doi=10.1061%2f9780784479865.051&partnerID=40&md5=52dd339e6306125e82d0ecaaf017cf5f
337	Viitanen, M. and Koivula, A. and Lemmetti, A. and Ylä-Outinen, A. and Vanne, J. and Hämäläinen, T.D.	Kvazaar: Open-source HEVC/H.265 encoder	10.1145/2964284.2973796	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84994682412&doi=10.1145%2f2964284.2973796&partnerID=40&md5=a2785ab8b43ac911b25242dd24be28af
338	Tsuji, J. and Weng, Z.	DNApi: A de novo adapter prediction algorithm for small RNA sequencing data	10.1371/journal.pone.0164228	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84991396410&doi=10.1371%2fjournal.pone.0164228&partnerID=40&md5=9d531f6a1e8acd4f8a4096334bd0492c
339	Hilton, M. and Tunnell, T. and Huang, K. and Marinov, D. and Dig, D.	Usage, costs, and benefits of continuous integration in open-source projects	10.1145/2970276.2970358	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84989159511&doi=10.1145%2f2970276.2970358&partnerID=40&md5=7e96113dc1efb43fb7b51e2d6ed14763
340	Jeanquartier, F. and Jean-Quartier, C. and Cemernek, D. and Holzinger, A.	In silico modeling for tumor growth visualization	10.1186/s12918-016-0318-8	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84984823845&doi=10.1186%2fs12918-016-0318-8&partnerID=40&md5=e702a59e0a2820987c79768921a8c28c
341	Cao, Y. and Wang, Y. and Zheng, X. and Li, F. and Bo, X.	RevEcoR: An R package for the reverse ecology analysis of microbiomes	10.1186/s12859-016-1088-4	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84984653407&doi=10.1186%2fs12859-016-1088-4&partnerID=40&md5=6edac424aeb4f6fd297f84fec3bcfa5e
342	Irawan, H. and Prihatmanto, A.S.	Implementation of graph database for OpenCog artificial general intelligence framework using Neo4j	10.1109/IDM.2015.7516355	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84983424321&doi=10.1109%2fIDM.2015.7516355&partnerID=40&md5=2fc001d89b1609b05cff4356607b6fea
343	Chapman, C. and Stolee, K.T.	Exploring regular expression usage and context in python	10.1145/2931037.2931073	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84984870747&doi=10.1145%2f2931037.2931073&partnerID=40&md5=138862ce9980e3c48f4949bb9a2802d5
344	Liu, H. and Hou, T.	CaFE: A tool for binding affinity prediction using end-point free energy methods	10.1093/bioinformatics/btw215	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84992346072&doi=10.1093%2fbioinformatics%2fbtw215&partnerID=40&md5=21d14d31b04eb690aa4a343898cf4e35
345	Liu, S. and Bai, G. and Sun, J. and Dong, J.S.	Towards Using Concurrent Java API Correctly	10.1109/ICECCS.2016.038	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85012098118&doi=10.1109%2fICECCS.2016.038&partnerID=40&md5=b11f347897f08bafbf4b0cffcc50328c
346	Brankovics, B. and Zhang, H. and van Diepeningen, A.D. and van der Lee, T.A.J. and Waalwijk, C. and de Hoog, G.S.	GRAbB: Selective Assembly of Genomic Regions, a New Niche for Genomic Research	10.1371/journal.pcbi.1004753	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84978828490&doi=10.1371%2fjournal.pcbi.1004753&partnerID=40&md5=8eb073a943810f7b63f2c416666264ba
347	Li, D. and Luo, R. and Liu, C.-M. and Leung, C.-M. and Ting, H.-F. and Sadakane, K. and Yamashita, H. and Lam, T.-W.	MEGAHIT v1.0: A fast and scalable metagenome assembler driven by advanced methodologies and community practices	10.1016/j.ymeth.2016.02.020	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84962227778&doi=10.1016%2fj.ymeth.2016.02.020&partnerID=40&md5=d27fc1f11afce0603dfc0af769b9c303
348	Chavarrías, C. and García-Vázquez, V. and Alemán-Gómez, Y. and Montesinos, P. and Pascau, J. and Desco, M.	fMRat: an extension of SPM for a fully automatic analysis of rodent brain functional magnetic resonance series	10.1007/s11517-015-1365-9	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84939479475&doi=10.1007%2fs11517-015-1365-9&partnerID=40&md5=ad0e61652d6f3d4d0fc739edb68f0c15
349	Cui, R. and Schumer, M. and Rosenthal, G.G.	Admix'em: A flexible framework for forward-time simulations of hybrid populations with selection and mate choice	10.1093/bioinformatics/btv700	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84964413487&doi=10.1093%2fbioinformatics%2fbtv700&partnerID=40&md5=afa9e2a98a720fa27c2e362e08db4621
350	Hu, H. and Wang, W. and Zhu, Z. and Zhu, J. and Tan, D. and Zhou, Z. and Mao, C. and Chen, X.	GIPS: A software guide to sequencing-based direct gene cloning in forward genetics studies	10.1104/pp.15.01327	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84962109512&doi=10.1104%2fpp.15.01327&partnerID=40&md5=2e80b8ed84361b97f98fcf9a9c907504
351	Basso, F. and Levorato, R. and Munaro, M. and Menegatti, E.	A distributed calibration algorithm for color and range camera networks	10.1007/978-3-319-26054-9_16	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84958581753&doi=10.1007%2f978-3-319-26054-9_16&partnerID=40&md5=1ad5bcba2616dec740eec236bfb6302b
352	Melo, A.T.O. and Bartaula, R. and Hale, I.	GBS-SNP-CROP: A reference-optional pipeline for SNP discovery and plant germplasm characterization using variable length, paired-end genotyping-by-sequencing data	10.1186/s12859-016-0879-y	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84953923814&doi=10.1186%2fs12859-016-0879-y&partnerID=40&md5=e7c81c8f469f3359475481f3b31271f4
354	Gilpin, W.	PyPDB: A Python API for the Protein Data Bank	10.1093/bioinformatics/btv543	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84959906715&doi=10.1093%2fbioinformatics%2fbtv543&partnerID=40&md5=7de012dd3ee8401ed67c9d9989534ec6
355	Busby, B. and Lesko, M. and Federer, L.	Closing gaps between open software and public data in a hackathon setting: User-centered software prototyping	10.12688/f1000research.8382.1	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84969640914&doi=10.12688%2ff1000research.8382.1&partnerID=40&md5=9dc49cba3aa5fdbb90ccb36fc84dd76f
357	Roe, K. and Smith, S.	Coqpie: An IDE aimed at improving proof development productivity	10.1007/978-3-319-43144-4_32	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84984815254&doi=10.1007%2f978-3-319-43144-4_32&partnerID=40&md5=b79e92dc29a5fa8a776e423e5096f798
358	Ambati, B.R. and Reddy, S. and Steedman, M.	Assessing relative sentence complexity using an incremental CCG parser	10.18653/v1/n16-1120	2016		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84994104623&doi=10.18653%2fv1%2fn16-1120&partnerID=40&md5=26475f1ec73ca1551895b3c3b1c1c3ea
362	Cui, X. and Wei, Z. and Zhang, L. and Liu, H. and Sun, L. and Zhang, S.-W. and Huang, Y. and Meng, J.	Sketching the distribution of transcriptomic features on RNA transcripts with Travis coordinates	10.1109/BIBM.2015.7359904	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84962427488&doi=10.1109%2fBIBM.2015.7359904&partnerID=40&md5=a102c11efa170fd77f2f4f4d62eebfd3
363	Nazarov, V.I. and Pogorelyy, M.V. and Komech, E.A. and Zvyagin, I.V. and Bolotin, D.A. and Shugay, M. and Chudakov, D.M. and Lebedev, Y.B. and Mamedov, I.Z.	tcR: An R package for T cell receptor repertoire advanced data analysis	10.1186/s12859-015-0613-1	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84939258340&doi=10.1186%2fs12859-015-0613-1&partnerID=40&md5=28f21836830cb6ba41f3c49af665f46e
364	Goeminne, M. and Mens, T.	Towards a survival analysis of database framework usage in Java projects	10.1109/ICSM.2015.7332512	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84961576026&doi=10.1109%2fICSM.2015.7332512&partnerID=40&md5=ad0d9afed762aac9df38356c3c88835a
365	Hoobin, C. and Kind, T. and Boucher, C. and Puglisi, S.J.	Fast and efficient compression of high-throughput sequencing reads	10.1145/2808719.2808753	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84963502761&doi=10.1145%2f2808719.2808753&partnerID=40&md5=4327d6eb60b44f35af988029f60c07d1
366	Vendome, C.	A Large Scale Study of License Usage on GitHub	10.1109/ICSE.2015.245	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84951802256&doi=10.1109%2fICSE.2015.245&partnerID=40&md5=56cc0cf822fd5c5a2737eec5f7cf7ffa
367	Vendome, C. and Linares-Vásquez, M. and Bavota, G. and Di Penta, M. and German, D. and Poshyvanyk, D.	License Usage and Changes: A Large-Scale Study of Java Projects on GitHub	10.1109/ICPC.2015.32	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84988267541&doi=10.1109%2fICPC.2015.32&partnerID=40&md5=c9a21d3e27caf9ead671b39820c29367
368	Sawant, A.A. and Bacchelli, A.	A dataset for API usage	10.1109/MSR.2015.75	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84957054673&doi=10.1109%2fMSR.2015.75&partnerID=40&md5=4985e472a4e2237225e39a700cf5865f
369	Forer, L. and Afgan, E. and Weißensteiner, H. and Davidović, D. and Specht, G. and Kronenberg, F. and Schönherr, S.	Cloudflow - A framework for MapReduce pipeline development in Biomedical Research	10.1109/MIPRO.2015.7160259	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84946119273&doi=10.1109%2fMIPRO.2015.7160259&partnerID=40&md5=531af452d43bf709a51218c0a0573798
370	Bickhart, D.M. and Hutchison, J.L. and Xu, L. and Schnabel, R.D. and Taylor, J.F. and Reecy, J.M. and Schroeder, S. and Van Tassell, C.P. and Sonstegard, T.S. and Liu, G.E.	RAPTR-SV: A hybrid method for the detection of structural variants	10.1093/bioinformatics/btv086	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84936760378&doi=10.1093%2fbioinformatics%2fbtv086&partnerID=40&md5=86cf8a447c4b858b6f37b03c9bc216af
371	De Klerk, E. and Fokkema, I.F.A.C. and Thiadens, K.A.M.H. and Goeman, J.J. and Palmblad, M. and Den Dunnen, J.T. and Von Lindern, M. and T'Hoen, P.A.C.	Assessing the translational landscape of myogenic differentiation by ribosome profiling	10.1093/nar/gkv281	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84936806742&doi=10.1093%2fnar%2fgkv281&partnerID=40&md5=869c2fedda5f34ffac38926154ed80b6
372	Bovy, J.	Galpy: A python library for galactic dynamics	10.1088/0067-0049/216/2/29	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84923246438&doi=10.1088%2f0067-0049%2f216%2f2%2f29&partnerID=40&md5=65aead4123f0657c19deee6cb32bb43d
373	Gnimpieba, E.Z. and Thavappiragasam, M. and Chango, A. and Conn, B. and Lushbough, C.M.	SBMLDock: Docker driven systems biology tool development and usage	10.1007/978-3-319-23401-4_24	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84945976513&doi=10.1007%2f978-3-319-23401-4_24&partnerID=40&md5=78f34c04aeec4a9f5354eba79305b6ca
374	Siranosian, B. and Perera, S. and Williams, E. and Ye, C. and de Graffenried, C. and Shank, P.	Tetranucleotide usage highlights genomic heterogeneity among mycobacteriophages	10.12688/f1000research.6077.2	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84969706679&doi=10.12688%2ff1000research.6077.2&partnerID=40&md5=9db859bbeccd0fb518cb90b5dd9d88e6
375	Silva, L.H. and Ramos, M. and Valente, M.T. and Bergel, A. and Anquetil, N.	Does JavaScript software embrace classes?	10.1109/SANER.2015.7081817	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84928666921&doi=10.1109%2fSANER.2015.7081817&partnerID=40&md5=ed9e27c2269b7ab9d3b17c21c413867d
376	Lawrence, T.J. and Kauffman, K.T. and Amrine, K.C.H. and Carper, D.L. and Lee, R.S. and Becich, P.J. and Canales, C.J. and Ardell, D.H.	FAST: FAST Analysis of Sequences Toolbox	10.3389/fgene.2015.00172	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84940103176&doi=10.3389%2ffgene.2015.00172&partnerID=40&md5=43fe4fe0cf4c7028be925d017e117ad7
377	Venev, S.V. and Zeldovich, K.B.	Massively parallel sampling of lattice proteins reveals foundations of thermal adaptation	10.1063/1.4927565	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84938798021&doi=10.1063%2f1.4927565&partnerID=40&md5=65045d1214bc544ac66adf3c8c9c167b
378	Almeida-E-Silva, D.C. and Vêncio, R.Z.N.	Sifter-T: A scalable and optimized framework for the SIFTER phylogenomic method of probabilistic protein domain annotation	10.2144/000114266	2015		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84924421750&doi=10.2144%2f000114266&partnerID=40&md5=8e1bcbbd919dc53a726e4efb747400b0
379	Zhang, Q. and Pell, J. and Canino-Koning, R. and Howe, A.C. and Brown, C.T.	These are not the K-mers you are looking for: Efficient online K-mer counting using a probabilistic data structure	10.1371/journal.pone.0101271	2014		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84904876565&doi=10.1371%2fjournal.pone.0101271&partnerID=40&md5=432e9e27fa984b14e51c1660f51d9e1b
380	Rosenberg, M.S.	Contextual cross-referencing of species names for fiddler crabs (genus Uca): An experiment in cyber-taxonomy	10.1371/journal.pone.0101704	2014		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84903888707&doi=10.1371%2fjournal.pone.0101704&partnerID=40&md5=4ec0029f1e5e1250d74057efeded7848
381	Billinger, M. and Brunner, C. and Müller-Putz, G.R.	SCoT: A python toolbox for EEG source connectivity	10.3389/fninf.2014.00022	2014		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84896924088&doi=10.3389%2ffninf.2014.00022&partnerID=40&md5=ca4fd24e321c184ba4562c5b60a0dcab
382	Bacci, G. and Bazzicalupo, M. and Benedetti, A. and Mengoni, A.	StreamingTrim 1.0: A Java software for dynamic trimming of 16S rRNA sequence data from metagenetic studies	10.1111/1755-0998.12187	2014		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84894243154&doi=10.1111%2f1755-0998.12187&partnerID=40&md5=296c000f43c62ef60c1366e6a0a2a38a
383	Rusk, D. and Coady, Y.	Location-based analysis of developers and technologies on GitHub	10.1109/WAINA.2014.110	2014		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84904480400&doi=10.1109%2fWAINA.2014.110&partnerID=40&md5=e976cddcd5b449f7fadf917c72e5d6f2
385	Bailey, D.H. and Borwein, J.M. and Kaiser, A.D.	Automated simplification of large symbolic expressions	10.1016/j.jsc.2013.09.001	2014		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84888197616&doi=10.1016%2fj.jsc.2013.09.001&partnerID=40&md5=db61f608d1e32bf86136cf0c75856d0b
386	Mostafa, S. and Wang, X.	An empirical study on the usage of mocking frameworks in software testing	10.1109/QSIC.2014.19	2014		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84912109692&doi=10.1109%2fQSIC.2014.19&partnerID=40&md5=7aa16634d3e1b305994c6a96e99d05c5
387	Padhye, R. and Mukherjee, D. and Sinha, V.S.	API as a social glue	10.1145/2591062.2591115	2014		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84903647935&doi=10.1145%2f2591062.2591115&partnerID=40&md5=6e78b05b3270e5948ff0b8c3086eae52
388	Turro, E. and Astle, W.J. and Tavaré, S.	Flexible analysis of RNA-seq data using mixed effects models	10.1093/bioinformatics/btt624	2014		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84892702660&doi=10.1093%2fbioinformatics%2fbtt624&partnerID=40&md5=d634563c63a5096766662673cac4163c
389	Krell, M.M. and Straube, S. and Seeland, A. and Wöhrle, H. and Teiwes, J. and Metzen, J.H. and Kirchner, E.A. and Kirchner, F.	PySPACE - A signal processing and classification environment in python	10.3389/fninf.2013.00040	2013		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84891553924&doi=10.3389%2ffninf.2013.00040&partnerID=40&md5=e1d801ea9c9c1ae1446def10bf897723
390	Teyton, C. and Falleri, J.-R. and Morandat, F. and Blanc, X.	Find your library experts	10.1109/WCRE.2013.6671295	2013		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84893360702&doi=10.1109%2fWCRE.2013.6671295&partnerID=40&md5=8e36ea130fa9089de62385d47ffadae4
391	McDonnell, T. and Ray, B. and Kim, M.	An empirical study of API stability and adoption in the android ecosystem	10.1109/ICSM.2013.18	2013		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84891682835&doi=10.1109%2fICSM.2013.18&partnerID=40&md5=230ac52147f4b89010f3c2bd8c417e4d
392	Itaya, H. and Oshita, K. and Arakawa, K. and Tomita, M.	GEMBASSY: An EMBOSS associated software package for comprehensive genome analyses	10.1186/1751-0473-8-17	2013		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84883514925&doi=10.1186%2f1751-0473-8-17&partnerID=40&md5=4fd33cdb4cc4df18893ca086bf8f9ada
393	Collet, P. and Lahire, P.	Feature modeling and separation of concerns with FAMILIAR	10.1109/CMA-RE.2013.6664179	2013		https://www.scopus.com/inward/record.uri?eid=2-s2.0-84892583365&doi=10.1109%2fCMA-RE.2013.6664179&partnerID=40&md5=546241a7c36bd4fecff0f834ace7491c
394	Regedor, M. and da Cruz, D. and Henriques, P.	The role of best practices to Appraise open source software	10.14279/tuj.eceasst.48.799.817	2012		https://www.scopus.com/inward/record.uri?eid=2-s2.0-85043292355&doi=10.14279%2ftuj.eceasst.48.799.817&partnerID=40&md5=8e1e0efeaaf5bdb320f626d9188aae7c
\.


--
-- Name: scopus_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval('public.scopus_id_seq', 394, true);


--
-- Name: scopus scopus_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
--

ALTER TABLE ONLY public.scopus
    ADD CONSTRAINT scopus_pkey PRIMARY KEY (id);


--
-- PostgreSQL database dump complete
--

